<article id="post-23325" class="et_pb_post post-23325 post type-post status-publish format-standard has-post-thumbnail hentry category-coronaviruses category-covid-19 category-disease-insight category-immunoassays category-new-products"><p><span>RT-PCR is the workhorse of viral diagnosis and has been invaluable in COVID-19 case confirmation and</span><span>&#160;isolation guidance. However, while fast and sensitive, PCR suffers from some inherent drawbacks that limit it to diagnosis during the acute phase of infection. To better understand the transmission dynamics of the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) and develop effective countermeasures against it, antigen- and antibody-based immunoassays will be essential. In this blog, we explain the key differences between PCR and immunoassays for COVID-19 diagnosis, and present our growing pipeline of coronavirus reagents for the development of highly accurate diagnostic kits.</span></p><p><span>Viral Diagnostics: A Quick Recap</span></p><p><span>The two major methods for diagnosing viral infection are the polymerase chain reaction (PCR), and immunoassays:</span></p><p><span><span>PCR</span></span><span> The polymerase chain reaction is a routine laboratory technique used to amplify small samples of DNA into larger quantities that can be detected and analysed. For patient diagnosis, a viral RNA or DNA sample is taken by swab or blood draw, before being sent to a specialist laboratory for analysis.</span></p><p><span>For RNA viruses, such as SARS-CoV-2, the polymerase chain reaction is preceded by an additional step to produce a complementary DNA template (cDNA) from RNA, by the addition of a reverse transcriptase enzyme (hence, Reverse Transcription-PCR (RT-PCR)).</span></p><p><span>PCR then begins by the addition of short DNA sequences known as primers, which bind the viral DNA strands. The double-stranded section of DNA is then recognised and bound by a thermostable polymerase enzyme which acts as a molecular photocopier to extend the sequence and produce a full, complementary strand.</span></p><p><span>Diagram showing the key stages of the polymerase chain reaction. Once generated, a cDNA template is added to a thermal cycler, along with free nucleotides, primers and Taq polymerase. The dsDNA is denatured with heat to create single strands. The temperature is then lowered to allow primers to anneal to complementary DNA sequences. By raising the temperature once more, the thermostable Taq polymerase can catalyse the addition of nucleotides to growing, complementary DNA strands.</span></p><p><span>By controlling the annealing, extension and denaturation steps with changes in temperature, the initial sample of viral DNA can be exponentially amplified, followed by the addition of specific DNA probes that produce a detectable signal (often fluorescent) to confirm the etiological agent.</span></p><p><span><span>Immunoassays</span> Unlike molecular techniques, immunoassays detect the presence of specific immune proteins. These assays take on a wide range of different formats, but essentially consist of an antigen or antibody, immobilised on a surface (most often a titre plate or paper strip), which binds virus-specific antigens or antibodies from a patient sample (i.e. sputum or blood sera). By adding a further reporter protein, it is then possible to detect a virus-specific immune signal to confirm the presence of ongoing or past viral infection.</span></p><p><span>Common examples of enzyme-linked immunosorbent assays (ELISAs). 1A) Antigen down ELISA. Titre plate is coated with antigen, which binds antigen-specific IgG antibodies from patient sera. Addition of a specific reporter antibody then confirms the presence of antigen-specific antibody; 1B) Double antigen bridging assay (DABA). Titre plate is coated with antigen, which binds antigen-specific IgG antibodies from patient sera. A second reporter antigen is then added to create an &#8216;antigen sandwich&#8217; ; 1C) Antibody capture ELISA. Titre plate is coated with specific antibody, which binds IgM from patient sera. Antigen is then added, before the addition of a reporter antibody to improve signal:noise. For more information on ELISA design, see our blog on <a href="https://thenativeantigencompany.com/elisa-kits/">ELISA formats</a>.</span></p><p><span>PCR for COVID-19</span></p><p><span>As a disease, COVID-19 shows considerable symptomatic variation. Individuals may present with no symptoms at all, while some may have a mild cough and fever, and an unlucky few may experience severe pneumonia and respiratory failure <a href="https://link.springer.com/article/10.1007/s11606-020-05762-w">[1]</a>.&#160;<span>To complicate matters, COVID-19&#8217;s most common symptoms strongly overlap with those presented by other co-circulating respiratory illnesses:</span></span></p><p><span>
<table id="tablepress-14" class="tablepress tablepress-id-14">
<thead>
<tr class="row-1">
	<th class="column-1">Symptom</th><th class="column-2">COVID-19</th><th class="column-3">Common Cold</th><th class="column-4">Flu</th><th class="column-5">Allergy</th>
</tr>
</thead>
<tbody>
<tr class="row-2">
	<td class="column-1">Fever</td><td class="column-2">Common</td><td class="column-3">Rare</td><td class="column-4">Common</td><td class="column-5">Occasional</td>
</tr>
<tr class="row-3">
	<td class="column-1">Dry Cough</td><td class="column-2">Common</td><td class="column-3">Mild</td><td class="column-4">Common</td><td class="column-5">Occasional</td>
</tr>
<tr class="row-4">
	<td class="column-1">Shortness of Breath</td><td class="column-2">Common</td><td class="column-3">No</td><td class="column-4">No</td><td class="column-5">Common</td>
</tr>
<tr class="row-5">
	<td class="column-1">Headache</td><td class="column-2">Occasional</td><td class="column-3">Rare</td><td class="column-4">Common</td><td class="column-5">Occasional</td>
</tr>
<tr class="row-6">
	<td class="column-1">Aches and Pains</td><td class="column-2">Occasional</td><td class="column-3">Common</td><td class="column-4">Common</td><td class="column-5">No
</td></tr>
<tr class="row-7">
	<td class="column-1">Sore Throat</td><td class="column-2">Occasional</td><td class="column-3">Common</td><td class="column-4">Common</td><td class="column-5">No
</td></tr>
<tr class="row-8">
	<td class="column-1">Fatigue</td><td class="column-2">Occasional</td><td class="column-3">Occasional</td><td class="column-4">Common</td><td class="column-5">Occasional</td>
</tr>
<tr class="row-9">
	<td class="column-1">Diarrhoea</td><td class="column-2">Common</td><td class="column-3">No</td><td class="column-4">Occasional</td><td class="column-5">No
</td></tr>
<tr class="row-10">
	<td class="column-1">Runny Nose</td><td class="column-2">Rare</td><td class="column-3">Common</td><td class="column-4">Occasional</td><td class="column-5">Common</td>
</tr>
<tr class="row-11">
	<td class="column-1">Sneezing</td><td class="column-2">No</td><td class="column-3">Common</td><td class="column-4">No</td><td class="column-5">Common</td>
</tr>
</tbody>
</table>
</span></p><p><span>Table comparing the typical symptoms of COVID-19, with the common cold, flu and allergy.</span></p><p><span><span>Because of this, diagnosing COVID-19 on the basis of clinical symptoms alone is highly inaccurate and must be confirmed by the use of highly specific diagnostic tests.&#160;</span></span><span>Fortunately, thanks to rapid sequencing and publication of the SARS-CoV-2 genome in early January</span><a href="https://www.ncbi.nlm.nih.gov/genbank/sars-cov-2-seqs/"> [2]</a><span>, RT-PCR primers and open access protocols were made&#160;</span><span>quickly</span><span> available, and are now being used by medical facilities worldwide to diagnose patients.</span></p><p><span>The main advantage of RT-PCR has been its speed and sensitivity. After taking a pharyngeal swab from the back of a patient&#8217;s throat, a sample can be sent to the lab to provide results within hours&#160;<a href="https://www.nature.com/articles/d41587-020-00010-2">[3]</a>. And as only a very small amount of viral RNA needs to be present for amplification, these tests are highly sensitive in detecting virus from a sample.</span></p><p><span><span>However, RT-PCR diagnosis of COVID-19 has its limitations. Detecting SARS-CoV-2 from pharyngeal swabs requires high-quality specimens that contain a sufficient amount of intact viral RNA. Yet,</span></span><span>&#160;SARS-CoV-2 loads in the respiratory tract have shown to vary considerably. This has not only led to high false-negative rates, with probable cases remaining negative after multiple swabs, but is further exposing healthcare workers to risk of infection</span><a href="https://www.nature.com/articles/d41587-020-00002-2">[3]</a><a href="https://www.preprints.org/manuscript/202003.0184/v1">[4]</a><span>.</span></p><p><span><span>There is also an issue of scale. Processing of COVID-19 samples requires specialised biocontainment laboratories, operated by highly trained technicians, usually only found within medium-large hospital facilities</span> <a href="https://hbr.org/2020/02/we-need-a-cheap-way-to-diagnose-coronavirus">[5]</a><span>. While capacity is normally sufficient, the COVID-19 pandemic is pushing these facilities to their limit.</span></span></p><p><span><span>For more resource-limited countries, diagnostic infrastructure and training is even harder to come by, with many healthcare systems having</span></span><span><span>&#160;only a handful of labs and technicians spread across vast geographical areas.&#160;</span></span><span>Governments and private organisations are working fast to increase the capacity and speed of PCR testing, but as laboratories are strained by growing case numbers, delays and complete lack of testing is becoming all-too-common </span><a href="https://www.vox.com/2020/3/20/21188266/coronavirus-test-us-united-states">[6]</a><span>.</span></p><p><span>Where do Immunoassays Fit In?</span></p><p><span>At a glance, immunoassays show some distinct advantages over PCR.&#160;</span><span>Antigens and antibodies are considerably more stable than RNA </span><span><a href="http://book.bionumbers.org/how-fast-do-rnas-and-proteins-degrade/">[7]</a>, which makes them less susceptible to spoliation during transport and storage, therefore reducing the chance of false-negative results</span><span>. Testing accuracy is also improved by the fact that antigens and antibodies are more uniformly available in sputum and blood samples.</span></p><p><span>However, the biggest advantage of immunoassays is their ability to detect past infections.</span></p><p><span>Once a patient has recovered from COVID-19 and virus is cleared from the body, viral RNA is no longer available for detection in the respiratory tract, leaving only a short window during&#160;the acute stage of infection in which SARS-CoV-2 can be detected.&#160;</span><span>While this works well for the diagnosis of ongoing infections, it gives no indication of whether a patient has had the infection historically, and what their immune status is (i.e. if they are immune to COVID-19 or still susceptible to infection).</span></p><p><span>Unlike RNA, antibodies are long-lasting and can persist in the bloodstream for many years after infection. As such, immunoassays enable us to identify patients that have had COVID-19, retrospectively. The type of antibody and its relative levels could also be used to indicate the stage of infection and estimate time since exposure for contact tracing.</span></p><p><span><span>However, antibody tests have their limitations too. As immunological data continues to emerge, it is becoming apparent that the body&#8217;s antibody response to COVID-19 is slow &#8211; considerably slower than we might expect. While data at this point is still limited, it appears that the initial IgM antibody response doesn&#8217;t peak until ~9 days after initial infection and the IgG antibody response doesn&#8217;t peak until day ~11 <a href="https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1">[8]</a><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.25727">[9]</a>.</span></span></p><p><span>Estimate of general biomarker levels during the typical timecourse of COVID-19/SARS-CoV-2 infection. Data from</span> <a href="https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1">Liu&#160;</a><a href="https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1">et al</a><a href="https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1">.</a> <span>and </span><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.25727">Li&#160;</a><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.25727">et al.</a> <span>Please note that this is purely illustrative and should not be used as a primary reference.</span></p><p><span>To put this into perspective, most viruses elicit a primary IgM response within 5 days&#160;<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3820351/">[10]</a>.&#160;</span><span>Because of this, SARS-CoV-2 antibodies are unlikely to make good markers of acute COVID-19 infection. While early studies have shown that combined RT-PCR/antibody testing can reduce false-negative rates</span><a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3546052">[11]</a><span>, the use of antibody kits for acute-phase diagnosis alone makes for a risky strategy.</span></p><p><span><span>So where does this leave antibody testing?</span></span></p><p><span><span>While antibodies may not be appropriate for acute-phase diagnosis, they still show many valuable applications for COVID-19.&#160;</span></span><span>Potentially the most valuable use of wide-scale antibody testing is as a public health tool.</span></p><p><span>To make the right public health decisions, governments worldwide are using estimates of transmission rates, case numbers and case fatality rates (CFR).&#160;</span><span>However, given</span><span><span>&#160;that&#160;anywhere from 20-80% of COVID-19 cases are estimated to be asymptomatic </span><a href="https://science.sciencemag.org/content/early/2020/03/13/science.abb3221">[12]</a><span>, these figures have been very difficult to accurately model.&#160;</span></span><span><span>On a population level, this means that the</span></span><span>&#160;true size and scope of the pandemic is still undefined, leaving policy makers with little indication of how serious of a threat COVID-19 still is, and how long it can be expected to last.</span></p><p><span><span>By conducting random antibody sampling of the general public (known as a serosurvey), public health bodies could better estimate the true levels of exposure&#160;</span></span><span>and resulting population immunity.&#160;</span><span>For COVID-19, this would be a game-changer, as true transmission and CFRs could be calculated to forecast the intensity and longevity of the pandemic to direct decision-making. Furthermore, by identifying potential geographical &#8216;hot-spots&#8217;</span><span>&#160;of low population immunity, health systems could better allocate resources to prevent or manage transmission.</span></p><p><span>Testing at the Point of Care</span></p><p><span>While antibodies aren&#8217;t suitable for diagnosing active COVID-19 infections, antigens are likely detectable in the sputum from the onset of symptoms&#160;<a href="https://jcm.asm.org/content/42/7/2884.short">[13]</a>. What&#8217;s more, by combining antigen detection with a format that can provide rapid results for use at the point-of-care (PoC), community-based, or even door-to-door testing could soon become a reality &#8211; allowing those who suspect an infection to get quick results and determine whether they should isolate and/or seek medical care.</span></p><p><span>The major caveat of formats such as ELISA, are that they are&#160;<span color="#000000">unwieldy, and require&#160;</span>trained scientists, operating in sterile laboratory conditions &#8211; not to mention, the complications of<span><span color="#000000" size="4">&#160;specimen collection and cold-chain logistics, that use up scarce healthcare resources.</span></span></span></p><p><span color="#000000" size="4">Fortunately, a range of PoC formats have been developed with simplicity and portability in mind.&#160;</span><span>So-called lateral flow assays (LFAs) are the most popular choice. These simple and easy-to-produce devices rely on the property of fluid analytes to move through absorbent substances on their own accord, via a phenomenon known as capillary flow.</span></p><p><span>Basic design of lateral flow assay, akin to home pregnancy tests. LFAs typically comprise a plastic cassette, containing a strip of paper that is able to absorb and transport analyte. 1) Analyte from blood or sputum is first added to one end of the paper strip, where it is absorbed by the paper, allowing the viral antigens to migrate along it. 2) These antigens first encounter an area of conjugate antibodies, which adhere to them, along with detectable tags. 3) The antigens then migrate to an area of test antibodies that are highly specific to the virus and bind to produce a solid, visible line that indicates a positive result, and hence, confirmed infection. Secondary control lines can then be used to detect conjugate antibodies and ensure that the test is working correctly.</span></p><p><span>These tests would be invaluable in providing rapid diagnosis in local communities, to give both healthcare systems and public health bodies a much clearer picture of unfolding pandemic.</span></p><p><span>Once governments are able to start reducing transmission, and COVID-19 case numbers begin to fall, PoC tests will become even more crucial in identifying asymptomatic carriers and&#160;<span>infected individuals</span> to ensure they are isolated from the general population.</span></p><p><span>While these kits are not currently available for widespread use, both public and private organisations, worldwide are working on prototypes, with over 50 currently in development</span> <span><a href="https://www.finddx.org/covid-19/pipeline/">[14]</a><span>.</span></span></p><p><span>Developing a Vaccine</span></p><p><span><span>So far, there are over 40 vaccine candidates for COVID-19 currently in the pipeline, with Moderna Therapeutic&#8217;s front-runner set to start human trials as early as next month <a href="https://investors.modernatx.com/news-releases/news-release-details/moderna-ships-mrna-vaccine-against-novel-coronavirus-mrna-1273">[15]</a>. While these vaccines are based on a wide range of platforms, (including mRNA, DNA, nanoparticles, subunits, synthetic peptides and virus-like particles, to name a few</span> <a href="https://www.who.int/blueprint/priority-diseases/key-action/novel-coronavirus-landscape-ncov.pdf?ua=1">[16]</a><span>), i</span></span><span>t can be said with near certainty, that a SARS-CoV-2 vaccine will elicit immune responses to the Spike protein.</span></p><p><span><span>Spike proteins are large, type I transmembrane proteins&#160;that protrude from the surface of the coronavirus virion. These proteins are&#160;</span>immunodominant (i.e. induce the strongest immune response), with the ability to elicit high neutralising antibody titres (i.e. antibodies that bind to the virus and prevent it from entering cells)&#160;<a href="https://www.biorxiv.org/content/10.1101/2020.01.28.923011v1">[17]</a>.</span></p><p><span>Diagram of coronavirus, showing key antigens for the development of diagnostics and vaccines. For more information on the Spike protein, please see our <a href="https://thenativeantigencompany.com/novel-coronavirus-antigens-now-available/">blog on its structure and immunology</a>.</span></p><p><span>As such, administration of a successful SARS-CoV-2 vaccine will induce anti-Spike IgG and IgM antibody responses to confer protection. However, serologically-speaking, vaccine-induced antibodies may be indistinguishable from natural infection.</span></p><p><span>This creates a headache for vaccine manufacturers, who need to be able to differentiate and measure vaccine-specific immune responses to prove efficacy during clinical trials.&#160;</span><span>Therefore, alternative antigens will need to be used when designing assays, to be able to differentiate between v</span><span>accine-induced and natural immune responses.</span></p><p><span>While there are multiple antigens to choose from, Nucleoprotein is a strong contender. <span>The Nucleoprotein is a vital structural protein with the primary function of forming a complex with viral RNA to mediate packaging and replication. While not as immunodominant as Spike, it is highly immunogenic and is profusely over-expressed during infection <a href="https://pubmed.ncbi.nlm.nih.gov/12584316/">[18]</a>.</span></span></p><p><span><span>The challenge in using Nucleoprotein, however, is its similarity between coronavirus strains.&#160;</span></span><span>Unlike Spike, Nucleoprotein&#160;shows less genetic variation &#8211; especially between SARS-CoV-2 and the genetically distinct, 2002 SARS Coronavirus.&#160;</span><span>The problem here, is that antibodies to SARS-CoV Nucleoprotein have the potential to cross-react with SARS-CoV-2 Nucleoprotein (i.e. the same antibodies are able to bind Nucleoprotein from both coronaviruses). Indeed, studies have shown the coronavirus Nucleoprotein to be</span><span>&#160;broadly cross-reactive <a href="https://www.nature.com/articles/s41586-020-2012-7">[19]</a></span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC404591/">[20]</a><span>.</span></p><p><span>Consequently, immunoassay developers will need to thoroughly screen the SARS-CoV-2 Nucleoprotein for antibody binding, and consider designing innovative immunoassay formats to avoid generating false-positive results.</span></p><p><span>Pathway of coronavirus pathogenesis, showing the point at which neutralising antibodies prevent viral entry via Angiotensin-Converting Enzyme 2 (ACE2).</span></p><p><span>The Convalescent Sera Option</span></p><p><span>With a vaccine still some time away, intermediary measures are being explored to protect those most at risk of severe COVID-19 infection. One of these is passive antibody therapy &#8211; a technique in which blood is taken from a recovered patient and purified for administration to protect or treat individuals.</span></p><p><span>Extraction, purification and administration of convalescent patient sera as a COVID-19 therapy. Adapted from <a href="https://www.jci.org/articles/view/138003">Casadevall and Pirofski</a>.</span></p><p><span>The antibodies conferred from the donor&#8217;s blood act as a stop-gap to provide immediate immunity while the individual develops their own humoral response to fight the infection</span><span><span>.&#160;</span><span>The main mechanism for this is through the action of neutralising antibodies, but also includes <a href="https://en.wikipedia.org/wiki/Antibody-dependent_cellular_cytotoxicity">antibody-dependent cellular cytotoxicity</a> (ADCC) and <a href="https://en.wikipedia.org/wiki/Antibody_opsonization">antibody-mediated phagocytosis</a></span><span>.</span></span></p><p><span>Passive antibody therapy dates back to the 1918 influenza pandemic, and was used with great effect during the recent 2009 H1N1 Influenza and 2013/2014 Ebola epidemics, showing significantly longer survival in treated patients&#160;<a href="https://www.jci.org/articles/view/138003">[21]</a>.&#160;</span><span>For COVID-19, antibodies could be used both prophylactically, to protect healthcare workers in COVID-19 wards, as well as a therapeutic for patients with severe infection.</span></p><p><span>To develop viable sera, patients&#8217; blood will be need to be screened for anti-SARS-CoV-2 antibodies (preferably neutralising).&#160;</span><span>This is most commonly done with so-called plaque neutralisation assays (PRNT). However, PRNT throughput is limited and needs to be carried out in specialised biocontainment facilities.&#160;</span><span>Therefore, immunoassays will likely have a crucial role to play in the qualitative and quantitative screening of convalescent sera.</span></p><p class="et_pb_text_inner"><p><span color="#0e5391"><span>Coronavirus Antigens and Antibodies from The Native Antigen Company</span></span>  </p>
<p><span color="#000000"><span>The Native Antigen Company offers an extensive and growing range of reagents for the <a href="https://thenativeantigencompany.com/human-coronavirus/">human coronaviruses</a>, including the emergent SARS-2, SARS, MERS coronaviruses, as well as the OC43, HKU1, 229E, and NL63 endemic coronaviruses.</span></span></p>
<p><span color="#000000"><span>In February 2020, The Native Antige<span>n Company became one of the world&#8217;s first suppliers to release <a href="https://thenativeantigencompany.com/novel-coronavirus-antigens-now-available/">commercially-available SARS-CoV-2 antigens</a>. Since then, our scientists have been hard at work, extending our coronavirus range to support the research and development of urgently-needed diagnostics and vaccines.</span></span></span></p>
<p><span>Our antigens are produced using our proprietary, </span><a href="https://thenativeantigencompany.com/about-us/">VirtuE</a><span>&#160;(HEK293) expression system, which ensures full glycosylation and proper folding, to ensure full biological and antigenic activity. T</span><span>o see our product range, please click on the</span><span>&#160;<span>available products</span></span><span>&#160;below.&#160;<span>You can keep up-to-date with new product developments on our <a href="https://thenativeantigencompany.com/coronavirus-dashboard/">Coronavirus Pipeline page</a>.</span></span></p>
<p><span>If we are not currently developing something that you&#8217;re interested in, please get in touch at <a href="mailto:contact@thenativeantigencompany.com">contact@thenativeantigencompany.com</a>, as we may be able to develop it for you.</span></p>
<table id="tablepress-12" class="tablepress tablepress-id-12">
<thead>
<tr class="row-1">
	<th class="column-1">Virus</th><th class="column-2">S1</th><th class="column-3">S2</th><th class="column-4">Whole Spike</th><th class="column-5">Nucleoprotein</th><th class="column-6">S1 NTD</th><th class="column-7">S1 RBD</th>
</tr>
</thead>
<tbody>
<tr class="row-2">
	<td class="column-1">&#160;SARS-CoV&#160;</td><td class="column-2"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-3">Antigens in early development</td><td class="column-4">Antigens in early development</td><td class="column-5"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-6">Antigens in early development</td><td class="column-7">Nothing yet scheduled</td>
</tr>
<tr class="row-3">
	<td class="column-1">&#160;SARS-CoV-2 &#160;</td><td class="column-2"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-3"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-4">Antigens in late development</td><td class="column-5"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-6"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-7"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td>
</tr>
<tr class="row-4">
	<td class="column-1">&#160;MERS-CoV&#160;</td><td class="column-2"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-3">Nothing yet scheduled</td><td class="column-4">Nothing yet scheduled</td><td class="column-5">Antigens in early development</td><td class="column-6">Nothing yet scheduled</td><td class="column-7">Nothing yet scheduled</td>
</tr>
<tr class="row-5">
	<td class="column-1">&#160;OC43-CoV&#160;</td><td class="column-2">Antigens in early development</td><td class="column-3">Nothing yet scheduled</td><td class="column-4">Antigens in early development</td><td class="column-5">Antigens in early development</td><td class="column-6">Nothing yet scheduled</td><td class="column-7">Nothing yet scheduled</td>
</tr>
<tr class="row-6">
	<td class="column-1">&#160;HKU1-CoV&#160;</td><td class="column-2">Antigens in early development</td><td class="column-3">Nothing yet scheduled</td><td class="column-4">Antigens in early development</td><td class="column-5">Antigens in early development</td><td class="column-6">Nothing yet scheduled</td><td class="column-7">Nothing yet scheduled</td>
</tr>
<tr class="row-7">
	<td class="column-1">&#160;229E-CoV&#160;</td><td class="column-2">Antigens in early development</td><td class="column-3">Nothing yet scheduled</td><td class="column-4">Antigens in early development</td><td class="column-5"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-6">Nothing yet scheduled</td><td class="column-7">Nothing yet scheduled</td>
</tr>
<tr class="row-8">
	<td class="column-1">&#160;NL63-CoV&#160;</td><td class="column-2">Antigens in early development</td><td class="column-3">Nothing yet scheduled</td><td class="column-4">Antigens in early development</td><td class="column-5"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antigens/">Antigens available</a></td><td class="column-6">Nothing yet scheduled</td><td class="column-7">Nothing yet scheduled</td>
</tr>
</tbody>
</table>
 
<table id="tablepress-13" class="tablepress tablepress-id-13">
<thead>
<tr class="row-1">
	<th class="column-1">Virus</th><th class="column-2">S1</th><th class="column-3">S2</th><th class="column-4">Whole Spike</th><th class="column-5">Nucleoprotein</th><th class="column-6">S1 NTD</th>
</tr>
</thead>
<tbody>
<tr class="row-2">
	<td class="column-1">&#160;SARS-CoV &#160;</td><td class="column-2">Antibodies in early development</td><td class="column-3"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antibodies/">Antibodies available</a></td><td class="column-4">Nothing yet scheduled</td><td class="column-5"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antibodies/">Antibodies available</a></td><td class="column-6">Nothing yet scheduled</td>
</tr>
<tr class="row-3">
	<td class="column-1">&#160;SARS-CoV-2&#160;</td><td class="column-2"><a href="https://thenativeantigencompany.com/products/human-igg1-anti-sars-cov-2-spike-s1-antibody-cr3022/">Antibodies available</a></td><td class="column-3"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antibodies/">Antibodies available</a></td><td class="column-4">Nothing yet scheduled</td><td class="column-5"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antibodies/">Antibodies available</a></td><td class="column-6">Nothing yet scheduled</td>
</tr>
<tr class="row-4">
	<td class="column-1">&#160;MERS-CoV&#160;</td><td class="column-2"><a href="https://thenativeantigencompany.com/product-category/all/antigens-and-antibodies/human-coronavirus-antibodies/">Antibodies available</a></td><td class="column-3">Nothing yet scheduled</td><td class="column-4">Nothing yet scheduled</td><td class="column-5">Nothing yet scheduled</td><td class="column-6">Nothing yet scheduled</td>
</tr>
</tbody>
</table>
</p><p><span color="#0e5391"><span>References</span></span>  </p><p><span color="#000000">1.&#160;<a href="https://link.springer.com/article/10.1007/s11606-020-05762-w">https://link.springer.com/article/10.1007/s11606-020-05762-w</a></span></p><p><span>2.&#160;<a href="https://www.ncbi.nlm.nih.gov/genbank/sars-cov-2-seqs/">https://www.ncbi.nlm.nih.gov/genbank/sars-cov-2-seqs/</a></span></p><p><span>3.&#160;<a href="https://www.nature.com/articles/d41587-020-00010-2">https://www.nature.com/articles/d41587-020-00010-2</a></span></p><p><span>4.&#160;<a href="https://www.preprints.org/manuscript/202003.0184/v1">https://www.preprints.org/manuscript/202003.0184/v1</a></span></p><p><span>5.&#160;<a href="https://hbr.org/2020/02/we-need-a-cheap-way-to-diagnose-coronavirus">https://hbr.org/2020/02/we-need-a-cheap-way-to-diagnose-coronavirus</a></span></p><p><span>6.&#160;<a href="https://www.vox.com/2020/3/20/21188266/coronavirus-test-us-united-states">https://www.vox.com/2020/3/20/21188266/coronavirus-test-us-united-states</a></span></p><p><span>7.&#160;<a href="http://book.bionumbers.org/how-fast-do-rnas-and-proteins-degrade/">http://book.bionumbers.org/how-fast-do-rnas-and-proteins-degrade/</a></span></p><p><span>8.&#160;<a href="https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1">https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1</a></span></p><p><span>9.&#160;<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.25727">https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.25727</a></span></p><p><span>10.&#160;<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3820351/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3820351/</a></span></p><p><span>11.&#160;<a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3546052">https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3546052</a></span></p><p><span>12.&#160;<a href="https://science.sciencemag.org/content/early/2020/03/13/science.abb3221">https://science.sciencemag.org/content/early/2020/03/13/science.abb3221</a></span></p><p><span>13.&#160;<a href="https://jcm.asm.org/content/42/7/2884.short">https://jcm.asm.org/content/42/7/2884.short</a></span></p><p><span>14.&#160;<a href="https://www.finddx.org/covid-19/pipeline/">https://www.finddx.org/covid-19/pipeline/</a></span></p><p><span>15.&#160;<a href="https://investors.modernatx.com/news-releases/news-release-details/moderna-ships-mrna-vaccine-against-novel-coronavirus-mrna-1273">https://investors.modernatx.com/news-releases/news-release-details/moderna-ships-mrna-vaccine-against-novel-coronavirus-mrna-1273</a></span></p><p><span>16.&#160;<a href="https://www.who.int/blueprint/priority-diseases/key-action/novel-coronavirus-landscape-ncov.pdf?ua=1">https://www.who.int/blueprint/priority-diseases/key-action/novel-coronavirus-landscape-ncov.pdf?ua=1</a></span></p><p><span>17.&#160;<a href="https://www.biorxiv.org/content/10.1101/2020.01.28.923011v1">https://www.biorxiv.org/content/10.1101/2020.01.28.923011v1</a></span></p><p><span>18.&#160;<a href="https://pubmed.ncbi.nlm.nih.gov/12584316/">https://pubmed.ncbi.nlm.nih.gov/12584316/</a></span></p><p><span>19.&#160;<a href="https://www.nature.com/articles/s41586-020-2012-7">https://www.nature.com/articles/s41586-020-2012-7</a></span></p><p><span>20.&#160;<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC404591/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC404591/</a></span></p><p><span>21.&#160;<a href="https://www.jci.org/articles/view/138003">https://www.jci.org/articles/view/138003</a></span></p></article>