<main class="shingle-content"><p class="wrap">
        <h1>Genetic Sequences for the SARS-CoV-2 Coronavirus</h1>
        
          <p class="lead">Nucleotide sequences of the SARS-CoV-2 virus (the virus associated with the COVID-19 disease, formerly known as 2019-nCoV) including location, collection time and similar supporting data.  (This data was imported and made computable at 8 am CDT on July 16, 2020.)  </p>
        
        
        
          
 
  
   <h2>Details</h2>
   <p class="notes">This data is imported from the National Center for Biotechnology Information (NCBI) and formatted for computation.</p>
   <p class="notes">Properties provided with each sequence include: &#8220;Accession&#8221;, &#8220;Length&#8221;, &#8220;Authors&#8221;, &#8220;Publications&#8221;, &#8220;GeographicLocation&#8221;, &#8220;DetailedGeographicLocation&#8221;, &#8220;Host&#8221;, &#8220;Sequence&#8221;, &#8220;CollectionDate&#8221;, &#8220;ReleaseDate&#8221;, &#8220;InclusionDate&#8221;, &#8220;SequenceType&#8221;, &#8220;NucleotideStatus&#8221;, &#8220;GenBankTitle&#8221;, &#8220;IsolationSource&#8221; and &#8220;BioSample&#8221;.</p>
   <p class="notes">Additional content elements include:</p>
   <p class="table-notes">
    <table class="grid">
     <tr>
      <td>
       <span class="table-text">"CollectionHistogram"</span>
      </td>
      <td>
       <span class="table-text">a <span class="inline-formula"><a class="reflink" href="https://reference.wolfram.com/language/ref/DateHistogram.html"><span class="symbol">DateHistogram</span></a></span> of when the sequences were collected</span>
      </td>
     </tr>
     <tr>
      <td>
       <span class="table-text">"ReleaseHistogram"</span>
      </td>
      <td>
       <span class="table-text">a <span class="inline-formula"><a class="reflink" href="https://reference.wolfram.com/language/ref/DateHistogram.html"><span class="symbol">DateHistogram</span></a></span> of when the sequences were released to the public</span>
      </td>
     </tr>
     <tr>
      <td>
       <span class="table-text">"InclusionHistogram"</span>
      </td>
      <td>
       <span class="table-text">a <span class="inline-formula"><a class="reflink" href="https://reference.wolfram.com/language/ref/DateHistogram.html"><span class="symbol">DateHistogram</span></a></span> of when the sequences were included in the source for this data</span>
      </td>
     </tr>
     <tr>
      <td>
       <span class="table-text">"AffectedLocations"</span>
      </td>
      <td>
       <span class="table-text">a world map showing where these sequences were collected</span>
      </td>
     </tr>
     <tr>
      <td>
       <span class="table-text">"AlignmentDifferences"</span>
      </td>
      <td>
       <span class="table-text">a <span class="inline-formula"><a class="reflink" href="https://reference.wolfram.com/language/ref/Dataset.html"><span class="symbol">Dataset</span></a></span> containing alignment differences with the reference sequence</span>
      </td>
     </tr>
    </table>
   </p>
  
 

        
        
        
        
        <p class="col main">
          <p class="nb-collapse label example-heading" tabindex="0">
            <h2 id="Examples">Examples</h2>
             
          </p>
          
            <p id="example-notebook-buttons" class="notebook-dropdown frame">
              
              
                <ul class="notebook-dropdown list">
                  
                    <li><a href="https://www.wolframcloud.com/env/cc33e40a-6286-4f3c-8d13-6fff7b1e2702?src=CloudBasicCopiedContent#sidebar=basic-notebook-links">Open in Cloud</a></li>
                  
                  
                    <li><a href="http://www.wolframcloud.com/files/cc33e40a-6286-4f3c-8d13-6fff7b1e2702?contentDisposition=attachment">Download Notebook</a></li>
                  
                </ul>
              
            </p>
          
          
             
            <h3>Basic Examples</h3><p class="example-text">Get a <span class="inline-formula"><a class="reflink" href="https://reference.wolfram.com/language/ref/Dataset.html"><span class="symbol">Dataset</span></a></span> containing rows for each sequence:</p><table class="example input"><tr><td class="in-out">In[1]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[1]=</td></tr></table><p class="example-text">Return the latest date a sequence was included:</p><table class="example input"><tr><td class="in-out">In[2]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[2]=</td></tr></table><p class="example-text">Count the different lengths of sequences provided, which corresponds well to the part of the virus that was sequenced:</p><table class="example input"><tr><td class="in-out">In[3]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[3]=</td></tr></table><p class="example-text">The lengths of sequences break down into two categories, corresponding to more complete sequences versus specific genetic regions:</p><table class="example input"><tr><td class="in-out">In[4]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[4]=</td></tr></table><p class="example-text">Most of these SARS-CoV-2 samples are collected from humans, but not all:</p><table class="example input"><tr><td class="in-out">In[5]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[5]=</td></tr></table><p class="subsection cell-group"><h3>Scope &amp; Additional Elements</h3><p class="example-text">Get a date histogram of collection dates:</p><table class="example input"><tr><td class="in-out">In[6]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[6]=</td></tr></table><p class="example-text">See a date histogram of release dates:</p><table class="example input"><tr><td class="in-out">In[7]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[7]=</td></tr></table><p class="example-text">See a date histogram of inclusion dates:</p><table class="example input"><tr><td class="in-out">In[8]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[8]=</td></tr></table><p class="example-text">Show the locations where the sequences were gathered:</p><table class="example input"><tr><td class="in-out">In[9]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[9]=</td></tr></table><p class="example-text">Obtain the available alignment differences with the reference sequence:</p><table class="example input"><tr><td class="in-out">In[10]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[10]=</td></tr></table></p><h3>Visualizations</h3><p class="example-text">A phylogenetic tree comparison of the most-common complete genomes by location shows clusters that are broadly distributed.  Dropping the trailing sequences of adenine terms avoids arbitrary differences from varying poly(A) RNA tail lengths, which may be sequencing artifacts and shouldn&amp;CloseCurlyQuote;t affect viral adaptivity:</p><table class="example input"><tr><td class="in-out">In[11]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[12]=</td></tr></table><table class="example input"><tr><td class="in-out">In[13]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[13]=</td></tr></table><p class="example-text">A similar visualization can be created for samples where more detailed geographic information is supplied. In this visualization of most-common sequences reported for US states, we see the emergence of clusters containing interesting regional blocks as shown in the map below:</p><table class="example input"><tr><td class="in-out">In[14]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[14]=</td></tr></table><table class="example input"><tr><td class="in-out">In[15]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[15]=</td></tr></table><p class="example-text">When visualizing the similarity of the most common sequence by week of sequence collection, while one does observe that similar times tend to cluster together, there is some overlap (such as between the week of Dec. 30, 2019 and the week of Feb. 17, 2020), illustrating that the virus has not only seen evolution, but significant continuity:</p><table class="example input"><tr><td class="in-out">In[16]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[17]=</td></tr></table><h3>Analysis</h3><p class="example-text">Using the provided alignment differences, we can see where along the viral genome changes have been detected over time.  We see that while mutations are relatively uniformly distributed, there are certainly changes more commonly measured than others:</p><table class="example input"><tr><td class="in-out">In[18]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[19]=</td></tr></table><p class="example-text">It is also possible to treat these genetic differences as lists of features:</p><table class="example input"><tr><td class="in-out">In[20]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[21]=</td></tr></table><p class="example-text">By doing so, it is possible to perform a fairly wide variety of analysis.  Here, we determine all of the genetic differences that always occur together in the sampled sequences, taking advantage of the fact that when differences always occur together they must occur in the same number of sequences:</p><table class="example input"><tr><td class="in-out">In[22]:=</td></tr></table><table class="example output"><tr><td class="in-out">Out[23]=</td></tr></table>
             
          
          
          
          
            <h2 id="External-Links">Bibliographic Citation</h2>
            <p>
              Wolfram Research,
              "Genetic Sequences for the SARS-CoV-2 Coronavirus"
              from the Wolfram Data Repository
              (2020)&#8194;<a href="https://doi.org/10.24097/wolfram.03304.data">https://doi.org/10.24097/wolfram.03304.data</a>
            </p>
          
          
            </p>
      </p></main>