<div><p class="container">
         
        For best viewing of the website please use Mozilla Firefox or Google Chrome.
    </p><p class="logowrap clear">
            Serving the Society Since 1986
             
             
        </p><ul class="fl clear">
                    <li type="Home"><a href="/index.htm">Home</a></li>
                                                                                <li type="AbouttheJournal">
                                                <a href="" target="_top">About the Journal</a>
                                                                        <ol class="qk-con nav-second data-show">
                                                                                    <li class="qk-con-p" type="AimsandScope">
                                                                <a href="/news/AimsandScope.htm" target="_top">Aims and Scope</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="EditorialBoard">
                                                                <a href="/news/EditorialBoard.htm" target="_top">Editorial Board</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="FormerEditorialBoard">
                                                                <a href="/news/FormerEditorialBoard.htm" target="_top">Former Editorial Board</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="EmailAlert">
                                                                <a href="/email_alert_en.htm" target="_top">Email Alert</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="Subscription">
                                                                <a href="/news/Subscription.htm" target="_top">Subscription</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="Contactus">
                                                                <a href="/news/Contactus.htm" target="_top">Contact us</a>
                                                            </li>
                                                                                </ol>
                                            </li>
                                                                                                                        <li type="browseJournal">
                                                <a href="" target="_top">Articles</a>
                                                                        <ol class="qk-con nav-second data-show">
                                                                                    <li class="qk-con-p" type="CurrentIssue">
                                                                <a href="/article/current" target="_top">Current Issue</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="JustAccepted">
                                                                <a href="/article/latest" target="_top">Just Accepted</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="Archive">
                                                                <a href="/archive_new_list_en.htm" target="_top">Archive</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="SpecialIssue">
                                                                <a href="/topic/topics" target="_top">Special Issue</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="FeaturedArticles">
                                                                <a href="/article/top_hot" target="_top">Featured Articles</a>
                                                            </li>
                                                                                </ol>
                                            </li>
                                                                                                                        <li type="ForAuthors">
                                                <a href="" target="_top">For Authors</a>
                                                                        <ol class="qk-con nav-second data-show">
                                                                                    <li class="qk-con-p" type="SubmitaManuscript">
                                                                <a href="https://mc03.manuscriptcentral.com/vs" target="_blank">Submit a Manuscript</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="InstructionsforAuthors">
                                                                <a href="/news/InstructionsforAuthors.htm" target="_top">Instructions for Authors</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="FastTrackPublication">
                                                                <a href="/news/FastTrackPublication.htm" target="_top">Fast Track Publication</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="EndnoteTemplate">
                                                                <a href="/news/EndnoteTemplate.htm" target="_top">Endnote Template</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="Ethics">
                                                                <a href="/news/Ethics.htm" target="_top">Ethics</a>
                                                            </li>
                                                                                </ol>
                                            </li>
                                                                                                                        <li type="ForReviewers">
                                                <a href="" target="_top">For Reviewers</a>
                                                                        <ol class="qk-con nav-second data-show">
                                                                                    <li class="qk-con-p" type="GuidelineforReviewers">
                                                                <a href="/news/GuidelineforReviewers.htm" target="_top">Guideline for Reviewers</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="Acknowledgements">
                                                                <a href="/news/Acknowledgements.htm" target="_top">Acknowledgements</a>
                                                            </li>
                                                                                </ol>
                                            </li>
                                                                                                                                                                                                                                                                                                                                <li type="NewsRelease">
                                                <a href="" target="_top">News Release</a>
                                                                        <ol class="qk-con nav-second data-show">
                                                                                    <li class="qk-con-p" type="JobNews">
                                                                <a href="/news_list_en.htm?column=JobNews" target="_top">Job News</a>
                                                            </li>
                                                                                                                <li class="qk-con-p" type="news">
                                                                <a href="/news_list_en.htm?column=news" target="_top">News</a>
                                                            </li>
                                                                                </ol>
                                            </li>
                                                                                <li><a href="/indexcn.htm" target="_blank">&#20013;&#25991;</a></li>
                </ul><p class="right fr clear">
                    
                        
                        
                        
                        
                        <a href="/to_advance_search" class="fr advanced">Advanced Search</a>
                    
                </p><ul class="smallUl">
                        <li type="AbouttheJournal">
                        <a href="" target="_top">About the Journal</a>
                                    <ol class="data-show">
                                <li type="AimsandScope">
                                        <a href="/news/AimsandScope.htm" target="_top">Aims and Scope</a>
                                    </li>
                                <li type="EditorialBoard">
                                        <a href="/news/EditorialBoard.htm" target="_top">Editorial Board</a>
                                    </li>
                                <li type="FormerEditorialBoard">
                                        <a href="/news/FormerEditorialBoard.htm" target="_top">Former Editorial Board</a>
                                    </li>
                                <li type="EmailAlert">
                                        <a href="/email_alert_en.htm" target="_top">Email Alert</a>
                                    </li>
                                <li type="Subscription">
                                        <a href="/news/Subscription.htm" target="_top">Subscription</a>
                                    </li>
                                <li type="Contactus">
                                        <a href="/news/Contactus.htm" target="_top">Contact us</a>
                                    </li>
                            </ol>
                    </li>
                                <li type="browseJournal">
                        <a href="" target="_top">Articles</a>
                                    <ol class="data-show">
                                <li type="CurrentIssue">
                                        <a href="/article/current" target="_top">Current Issue</a>
                                    </li>
                                <li type="JustAccepted">
                                        <a href="/article/latest" target="_top">Just Accepted</a>
                                    </li>
                                <li type="Archive">
                                        <a href="/archive_new_list_en.htm" target="_top">Archive</a>
                                    </li>
                                <li type="SpecialIssue">
                                        <a href="/topic/topics" target="_top">Special Issue</a>
                                    </li>
                                <li type="FeaturedArticles">
                                        <a href="/article/top_hot" target="_top">Featured Articles</a>
                                    </li>
                            </ol>
                    </li>
                                <li type="ForAuthors">
                        <a href="" target="_top">For Authors</a>
                                    <ol class="data-show">
                                <li type="SubmitaManuscript">
                                        <a href="https://mc03.manuscriptcentral.com/vs" target="_blank">Submit a Manuscript</a>
                                    </li>
                                <li type="InstructionsforAuthors">
                                        <a href="/news/InstructionsforAuthors.htm" target="_top">Instructions for Authors</a>
                                    </li>
                                <li type="FastTrackPublication">
                                        <a href="/news/FastTrackPublication.htm" target="_top">Fast Track Publication</a>
                                    </li>
                                <li type="EndnoteTemplate">
                                        <a href="/news/EndnoteTemplate.htm" target="_top">Endnote Template</a>
                                    </li>
                                <li type="Ethics">
                                        <a href="/news/Ethics.htm" target="_top">Ethics</a>
                                    </li>
                            </ol>
                    </li>
                                <li type="ForReviewers">
                        <a href="" target="_top">For Reviewers</a>
                                    <ol class="data-show">
                                <li type="GuidelineforReviewers">
                                        <a href="/news/GuidelineforReviewers.htm" target="_top">Guideline for Reviewers</a>
                                    </li>
                                <li type="Acknowledgements">
                                        <a href="/news/Acknowledgements.htm" target="_top">Acknowledgements</a>
                                    </li>
                            </ol>
                    </li>
                                                                                                                <li type="NewsRelease">
                        <a href="" target="_top">News Release</a>
                                    <ol class="data-show">
                                <li type="JobNews">
                                        <a href="/news_list_en.htm?column=JobNews" target="_top">Job News</a>
                                    </li>
                                <li type="news">
                                        <a href="/news_list_en.htm?column=news" target="_top">News</a>
                                    </li>
                            </ol>
                    </li>
                        <li><a href="/indexcn.htm">&#20013;&#25991;</a></li>
    </ul><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00124-1" class="tit" target="_blank">Binding between Prion Protein and A&#946; Oligomers Contributes to the Pathogenesis of Alzheimer's Disease</a>
                                                <p class="text">
                            A plethora of evidence suggests that protein misfolding and aggregation are underlying mechanisms of various neurodegenerative diseases, such as prion diseases and Alzheimer's disease (AD). Like prion diseases, AD has been considered as an infectious disease in the past decades as it shows strain specificity and transmission potential. Although it remains elusive how protein aggregation leads to AD, it is becoming clear that cellular prion protein (PrPC) plays an important role in AD pathogenesis. Here, we briefly reviewed AD pathogenesis and focused on recent progresses how PrPC contributed to AD development. In addition, we proposed a potential mechanism to explain why infectious agents, such as viruses, conduce AD pathogenesis. Microbe infections cause A&#946; deposition and upregulation of PrPC, which lead to high affinity binding between A&#946; oligomers and PrPC. The interaction between PrPC and A&#946; oligomers in turn activates the Fyn signaling cascade, resulting in neuron death in the central nervous system (CNS). Thus, silencing PrPC expression may turn out be an effective treatment for PrPC dependent AD.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00124-1" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00141-0" class="tit" target="_blank">SNX11 Identified as an Essential Host Factor for SFTS Virus Infection by CRISPR Knockout Screening</a>
                                                <p class="text">
                            Severe fever with thrombocytopenia syndrome virus (SFTSV) is a highly pathogenic tick-borne bunyavirus that causes lethal infectious disease and severe fever with thrombocytopenia syndrome (SFTS) in humans. The molecular mechanisms and host cellular factors required for SFTSV infection remain uncharacterized. Using a genome-wide CRISPR-based screening strategy, we identified a host cellular protein, sorting nexin 11 (SNX11) which is involved in the intracellular endosomal trafficking pathway, as an essential cell factor for SFTSV infection. An SNX11-KO HeLa cell line was established, and SFTSV replication was significantly reduced. The glycoproteins of SFTSV were detected and remained in later endosomal compartments but were not detectable in the endoplasmic reticulum (ER) or Golgi apparatus. pH values in the endosomal compartments of the SNX11-KO cells increased compared with the pH of normal HeLa cells, and lysosomal-associated membrane protein 1 (LAMP1) expression was significantly elevated in the SNX11-KO cells. Overall, these results indicated that penetration of SFTSV from the endolysosomes into the cytoplasm of host cells was blocked in the cells lacking SNX11. Our study for the first time provides insight into the important role of the SNX11 as an essential host factor in the intracellular trafficking and penetrating process of SFTSV infection via potential regulation of viral protein sorting, membrane fusion, and other endocytic machinery.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00141-0" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00120-5" class="tit" target="_blank">Mitochondria Redistribution in Enterovirus A71 Infected Cells and Its Effect on Virus Replication</a>
                                                <p class="text">
                            Enterovirus A71 (EV-A71) is one of the main causative agents of hand, foot and mouth disease (HFMD) and it also causes severe neurologic complications in infected children. The interactions between some viruses and the host mitochondria are crucial for virus replication and pathogenicity. In this study, it was observed that EV-A71 infection resulted in a perinuclear redistribution of the mitochondria. The mitochondria rearrangement was found to require the microtubule network, the dynein complex and a low cytosolic calcium concentration. Subsequently, the EV-A71 non-structural protein 2BC was identified as the viral protein capable of inducing mitochondria clustering. The protein was found localized on mitochondria and interacted with the mitochondrial Rho GTPase 1 (RHOT1) that is a key protein required for attachment between the mitochondria and the motor proteins, which are responsible for the control of mitochondria movement. Additionally, suppressing mitochondria clustering by treating cells with nocodazole, EHNA, thapsigargin or A23187 consistently inhibited EV-A71 replication, indicating that mitochondria recruitment played a crucial role in the EV-A71 life cycle. This study identified a novel function of the EV-A71 2BC protein and provided a potential model for the regulation of mitochondrial motility in EV-A71 infection.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00120-5" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00125-0" class="tit" target="_blank">Specific and Selective Bacteriophages in the Fight against Multidrugresistant Acinetobacter baumannii</a>
                                                <p class="text">
                            Acinetobacter baumannii causes serious infections especially in immunocompromised and/or hospitalized patients. Several A. baumannii strains are multidrug resistant and infect wounds, bones, and the respiratory tract. Current studies are focused on finding new effective agents against A. baumannii. Phage therapy is a promising means to fight this bacterium and many studies on procuring and applying new phages against A. baumannii are currently being conducted. As shown in animal models, phages against multidrug-resistant A. baumannii may control bacterial infections caused by this pathogen and may be a real hope to solve this dangerous health problem.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00125-0" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00111-6" class="tit" target="_blank">Effect of Loss-of-function of the Herpes Simplex Virus-1 microRNA H6-5p on Virus Replication</a>
                                                <p class="text">
                            To date, 29 distinct microRNAs (miRNAs) have been reported to be expressed during herpes simplex virus infections. Sequence analysis of mature herpes simplex virus-1 (HSV-1) miRNAs revealed five sets of miRNAs that are complementary to each other: miR-H6-5p/H1-3p, miR-H6-3p/H1-5p, H2-5p/H14-3p, miR-H2-3p/H14-5p, and miR-H7/H27. However, the roles of individual miRNAs and consequences of this complementarity remain unclear. Here, we focus on two of these complementary miRNAs, miR-H6-5p and miR-H1-3p, using loss-of-function experiments in vitro and in a mouse model of infection using an miRNA sponge approach, including tandem multiplex artificial miRNA-binding sequences that do not match perfectly to the target miRNA inserted downstream of a green fluorescent protein reporter gene. Infection with recombinant virus expressing the miR-H6-5p sponge reduced viral protein levels and virus yield. Decreased accumulation of viral proteins was also observed at early stages of infection in the presence of both an miR-H6-5p inhibitor and plasmid-expressed miR-H1-3p. Moreover, establishment of latency and reactivation did not differ between the recombinant virus expressing the miR-H6-5p sponge and wild-type HSV-1. Taken together, these data suggest that miR-H6-5p has an as-yet-unidentified role in the early stages of viral infection, and its complement miR-H1-3p suppresses this role in later stages of infection. This report extends understanding of the roles of miRNAs in infection by herpes simplex viruses, supporting a model of infection in which the production of virus and its virulent effects are tightly controlled to maximize persistence in the host and population.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00111-6" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0055-9" class="tit" target="_blank">MicroRNA-135a Modulates Hepatitis C Virus Genome Replication through Downregulation of Host Antiviral Factors</a>
                                                <p class="text">
                            Cellular microRNAs (miRNAs) have been shown to modulate HCV infection via directly acting on the viral genome or indirectly through targeting the virus-associated host factors. Recently we generated a comprehensive map of HCV&#8211; miRNA interactions through genome-wide miRNA functional screens and transcriptomics analyses. Many previously unappreciated cellular miRNAs were identified to be involved in HCV infection, including miR-135a, a human cancerrelated miRNA. In the present study, we investigated the role of miR-135a in regulating HCV life cycle and showed that it preferentially enhances viral genome replication. Bioinformatics-based integrative analyses and subsequent functional assays revealed three antiviral host factors, including receptor interacting serine/threonine kinase 2 (RIPK2), myeloid differentiation primary response 88 (MYD88), and C-X-C motif chemokine ligand 12 (CXCL12), as bona fide targets of miR-135a. These genes have been shown to inhibit HCV infection at the RNA replication stage. Our data demonstrated that repression of key host restriction factors mediated the proviral effect of miR-135a on HCV propagation. In addition, miR-135a hepatic abundance is upregulated by HCV infection in both cultured hepatocytes and human liver, likely mediating a more favorable environment for viral replication and possibly contributing to HCV-induced liver malignancy. These results provide novel insights into HCV&#8211;host interactions and unveil molecular pathways linking miRNA biology to HCV pathogenesis.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-018-0055-9" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00114-3" class="tit" target="_blank">Towards Better Understanding of KSHV Life Cycle: from Transcription and Posttranscriptional Regulations to Pathogenesis</a>
                                                <p class="text">
                            Kaposi's sarcoma-associated herpesvirus (KSHV), also known as human herpesvirus-8 (HHV-8), is etiologically linked to the development of Kaposi's sarcoma, primary effusion lymphoma, and multicentric Castleman's disease. These malignancies often occur in immunosuppressed individuals, making KSHV infection-associated diseases an increasing global health concern with persistence of the AIDS epidemic. KSHV exhibits biphasic life cycles between latent and lytic infection and extensive transcriptional and posttranscriptional regulation of gene expression. As a member of the herpesvirus family, KSHV has evolved many strategies to evade the host immune response, which help the virus establish a successful lifelong infection. In this review, we summarize the current research status on the biology of latent and lytic viral infection, the regulation of viral life cycles and the related pathogenesis.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00114-3" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00122-3" class="tit" target="_blank">Viral Regulation of RNA Granules in Infected Cells</a>
                                                <p class="text">
                            RNA granules are cytoplasmic, microscopically visible, non-membrane ribo-nucleoprotein structures and are important posttranscriptional regulators in gene expression by controlling RNA translation and stability. TIA/G3BP/PABP-specific stress granules (SG) and GW182/DCP-specific RNA processing bodies (PB) are two major distinguishable RNA granules in somatic cells and contain various ribosomal subunits, translation factors, scaffold proteins, RNA-binding proteins, RNA decay enzymes and helicases to exclude mRNAs from the cellular active translational pool. Although SG formation is inducible due to cellular stress, PB exist physiologically in every cell. Both RNA granules are important components of the host antiviral defense. Virus infection imposes stress on host cells and thus induces SG formation. However, both RNA and DNA viruses must confront the hostile environment of host innate immunity and apply various strategies to block the formation of SG and PB for their effective infection and multiplication. This review summarizes the current research development in the field and the mechanisms of how individual viruses suppress the formation of host SG and PB for virus production.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00122-3" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-019-00098-0" class="tit" target="_blank">HPV18 Utilizes Two Alternative Branch Sites for E6*I Splicing to Produce E7 Protein</a>
                                                <p class="text">
                            Human papillomavirus 18 (HPV18) E6 and E7 oncogenes are transcribed as a single bicistronic E6E7 pre-mRNA. The E6 ORF region in the bicistronic E6E7 pre-mRNA contains an intron. Splicing of this intron disrupts the E6 ORF integrity and produces a spliced E6*I RNA for efficient E7 translation. Here we report that the E6 intron has two overlapped branch point sequences (BPS) upstream of its 3' splice site, with an identical heptamer AACUAAC, for E6*I splicing. One heptamer has a branch site adenosine (underlined) at nt 384 and the other at nt 388. E6*I splicing efficiency correlates to the expression level of E6 and E7 proteins and depends on the selection of which branch site. In general, E6*I splicing prefers the 3'ss-proximal branch site at nt 388 over the distal branch site at nt 384. Inactivation of the nt 388 branch site was found to activate a cryptic acceptor site at nt 636 for aberrant RNA splicing. Together, these data suggest that HPV18 modulates its production ratio of E6 and E7 proteins by alternative selection of the two mapped branch sites for the E6*I splicing, which could be beneficial in its productive or oncogenic infection according to the host cell environment.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-019-00098-0" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0071-9" class="tit" target="_blank">Recent Progress on Functional Genomics Research of Enterovirus 71</a>
                                                <p class="text">
                            Enterovirus 71 (EV71) is one of the main pathogens that causes hand-foot-and-mouth disease (HFMD). HFMD caused by EV71 infection is mostly self-limited; however, some infections can cause severe neurological diseases, such as aseptic meningitis, brain stem encephalitis, and even death. There are still no effective clinical drugs used for the prevention and treatment of HFMD. Studying EV71 protein function is essential for elucidating the EV71 replication process and developing anti-EV71 drugs and vaccines. In this review, we summarized the recent progress in the studies of EV71 noncoding regions (5' UTR and 3' UTR) and all structural and nonstructural proteins, especially the key motifs involving in viral infection, replication, and immune regulation. This review will promote our understanding of EV71 virus replication and pathogenesis, and will facilitate the development of novel drugs or vaccines to treat EV71.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-018-0071-9" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0075-5" class="tit" target="_blank">Regulation of Virus Replication and T Cell Homeostasis by N6-Methyladenosine</a>
                                                <p class="text">
                            RNA modifications are abundant in eukaryotes, bacteria, and archaea. N6-methyladenosine (m6A), a type of RNA modification mainly found in messenger RNA (mRNA), has significant effects on the metabolism and function of mRNAs. This modification is governed by three types of proteins, namely methyltransferases as "writers", demethylases as "erasers", and specific m6A-binding proteins (YTHDF1-3) as "readers". Further, it is important for the regulation of cell fate and has a critical function in many biological processes including virus replication, stem cell differentiation, and cancer development, and exerts its effect by controlling gene expression. Herein, we summarize recent advances in research on m6A in virus replication and T cell regulation, which is a rapidly emerging field that will facilitate the development of antiviral therapies and the study of innate immunity.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-018-0075-5" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0078-2" class="tit" target="_blank">The Asian Lineage of Zika Virus: Transmission and Evolution in Asia and the Americas</a>
                                                <p class="text">
                            Since first isolation in 1947 from the Zika forest in Uganda, Zika virus (ZIKV) has been principally known as a benign agent associated with sporadic human infections in a restricted number of African countries. However, during 2015&#8211;2016, an Asian lineage of ZIKV caused an unprecedentedly large outbreak in the Americas and sizeable numbers of exported cases across the globe. In this review, we critically appraise the recent advances in molecular epidemiological studies of ZIKV performed to date, and we highlight the pivotal role played by genomic surveillance in elucidating the origins, dissemination and evolution of the Asian lineage of ZIKV in Asia and in the Americas.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-018-0078-2" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0076-4" class="tit" target="_blank">Genetic Evidence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) and Widespread Seroprevalence among Camels in Kenya</a>
                                                <p class="text">
                            We describe the first genome isolation of Middle East respiratory syndrome coronavirus (MERS-CoV) in Kenya. This fatal zoonotic pathogen was first described in the Kingdom of Saudi Arabia in 2012. Epidemiological and molecular evidence revealed zoonotic transmission from camels to humans and between humans. Currently, MERS-CoV is classified by the WHO as having high pandemic potential requiring greater surveillance. Previous studies of MERS-CoV in Kenya mainly focused on site-specific and archived camel and human serum samples for antibodies. We conducted active nationwide cross-sectional surveillance of camels and humans in Kenya, targeting both nasal swabs and plasma samples from 1, 163 camels and 486 humans collected from January 2016 to June 2018. A total of 792 camel plasma samples were positive by ELISA. Seroprevalence increased with age, and the highest prevalence was observed in adult camels (82.37%, 95% confidence interval (CI) 79.50-84.91). More female camels were significantly seropositive (74.28%, 95% CI 71.14-77.19) than male camels (P &lt; 0.001) (53.74%, 95% CI 48.48-58.90). Only 11 camel nasal swabs were positive for MERS-CoV by reverse transcription-quantitative PCR. Phylogenetic analysis of whole genome sequences showed that Kenyan MERS-CoV clustered within sub-clade C2, which is associated with the African clade, but did not contain signature deletions of orf4b in African viruses. None of the human plasma screened contained neutralizing antibodies against MERS-CoV. This study confirms the geographically widespread occurrence of MERS-CoV in Kenyan camels. Further one-health surveillance approaches in camels, wildlife, and human populations are needed.
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-018-0076-4" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0079-1" class="tit" target="_blank">On the Centenary of the Spanish Flu: Being Prepared for the Next Pandemic</a>
                                                <p class="text">
                            Influenza is one of the oldest infectious diseases affecting humans. Every influenza pandemic in history has ended with disastrous outcomes regarding public health and the social economy. This year marks the 100th anniversary of the Spanish flu (H1N1) outbreak of 1918, which is recognized as the most lethal natural event in recent history. In spite of limited travel and transportation at that time, the Spanish flu pandemic of 1918 reached peak activity on multiple continents simultaneously within several months after its emergence in late 1917 from different hypothesized origins, such as US military camps, the state of Kansas, or the troop staging and hospital camp in &#201;xtaples, France (Patterson and Pyle 1991; Oxford et al. 2005;  Shanks 2016). However, in some islands of the Pacific region, such as in New Caledonia, the pandemic's lethal effects lasted for over 3 years, until July 1921 (Shanks et al. 2018). The pandemic flu is estimated to have infected more than 500 million people, causing between 50 and 100 million deaths globally (Patterson and Pyle 1991; Guan et al. 2010). The evidence suggests that most pandemicrelated deaths were not caused by the direct pathological effects of the influenza virus, but rather by the lethal effects of secondary bacterial pneumonia (Chien et al. 2009).
                        </p>
                        		 					<a href="/article/doi/10.1007/s12250-018-0079-1" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><p class="in-bl clear">
						 					 
                    <p class="right fl">
                   				 					<a href="/article/doi/10.1007/s12250-018-0050-1" class="tit" target="_blank">Phylogeography, Transmission, and Viral Proteins of Nipah Virus</a>
                                                <p class="text">
                            Nipah virus (NiV), a zoonotic paramyxovirus belonging to the genus&#160;Henipavirus, is classified as a Biosafety Level-4 pathogen based on its high pathogenicity in humans and the lack of available vaccines or therapeutics. Since its initial emergence in 1998 in Malaysia, this virus has become a great threat to domestic animals and humans. Sporadic outbreaks and person-to-person transmission over the past two decades have resulted in hundreds of human fatalities. Epidemiological surveys have shown that NiV is distributed in Asia, Africa, and the South Pacific Ocean, and is transmitted by its natural reservoir,&#160;Pteropid&#160;bats. Numerous efforts have been made to analyze viral protein function and structure to develop feasible strategies for drug design. Increasing surveillance and preventative measures for the viral infectious disease are urgently needed.</p> 
                        		 					<a href="/article/doi/10.1007/s12250-018-0050-1" class="more" target="_blank">more &gt;&gt;</a>
                                            </p>
                </p><ul id="myTab" class="tab-ul clearfix">
            <li class="current_ active"><a href="#current">Current Issue</a></li>
            <li><a href="#onlineFirst">Just Accepted</a></li>
            <li><a href="/topic/topics">Special Issues</a></li>
        </ul><p class="priorityPrompt colorRed">Articles in press have been peer-reviewed and accepted, which are not yet assigned to volumes/issues, but are citable by Digital Object Identifier (DOI).</p><p class="article-list-right clear">
				       	  
    				
				<p class="article-list-title">
	           										               		<a href="article/id/d9eabaac-5c75-45bd-b510-b2fca8d4949a" class="mainColor" target="_blank">A Small-Scale Medication of Leflunomide as a Treatment of COVID-19 in an Open-Label Blank-Controlled Clinical Trial</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Ke Hu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mengmei Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yang Zhao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yunting Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Tao Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhishui Zheng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiaochen Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Shaolin Zeng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Dong Zhao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Honglin Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ke Xu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ke Lan</a>	                    </span>
	                				</p>
				
			
		         
		      	<p class="article-list-date">
            		            	
	            		Received: <span>03 June 2020</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/id/d9eabaac-5c75-45bd-b510-b2fca8d4949a" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/%24%7Bv.doi%7D" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">We recently reported that inhibitors against human dihydroorotate dehydrogenase (DHODH) have broad-spectrum antiviral activities including their inhibitory efficacies on SARS-CoV-2 replication in infected cells. However, there are limited data from clinical studies to prove the application of DHODH inhibitors in Coronavirus Disease 2019 (COVID-19) patients. In present study, we evaluated Leflunomide, an approved DHODH inhibitor widely used as a modest immune regulator to treat autoimmune diseases, in treating COVID-19 disease with a small-scale of patients. Cases of 10 laboratory-confirmed COVID-19 patients of moderate type with obvious opacity in the lung were included. Five of the patients were treated with Leflunomide, and another five were treated as blank controls without a placebo. All the patients accepted standard supportive treatment for COVID-19. The patients given Leflunomide had a shorter viral shedding time (median of 5 days) than the controls (median of 11 days, P=0.046). The patients given Leflunomide also showed a significant reduction in C-reactive protein levels, indicating that immunopathological inflammation was well controlled. No obvious adverse effects were observed in Leflunomide-treated patients, and they all discharged from the hospital faster than controls. This preliminary study on a small-scale compassionate use of Leflunomide provides clues for further understanding of Leflunomide as a potential antiviral drug against COVID-19.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00230-5" class="mainColor" target="_blank">On the Calculation of TCID50 for Quantitation of Virus Infectivity</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Chengfeng Lei</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jian Yang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jia Hu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiulian Sun</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00230-5" target="_blank" class="list-doi">10.1007/s12250-020-00230-5</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>10 March 2020</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00230-5" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00230-5" target="_blank">Springerlink</a>        
      
                 
                       <a href="article/doi/10.1007/s12250-020-00230-5" target="_blank">ESM</a>
                    
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">The most important property of a virus is its infectivity. To measure infectivity, one can assay viral replication in cells to obtain a titer for a given virus stock. A titer is defined as a given number of infectious viral units per unit volume, and an infectious unit is the smallest amount of virus that produces recognizable effects [e.g., cytopathic effect (CPE), dot blot immunoreactivity]. The median tissue culture infectious dose (TCID50) is defined as the dilution of a virus required to infect 50% of a given cell culture.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00228-z" class="mainColor" target="_blank">First Fatal Infection and Phylodynamic Analysis of Severe Fever with Thrombocytopenia Syndrome Virus in Jilin Province, Northeastern China</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Xu Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Nina Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zedong Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lihe Che</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Chen Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wen-Zhong Zhao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Quan Liu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00228-z" target="_blank" class="list-doi">10.1007/s12250-020-00228-z</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>30 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00228-z" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00228-z" target="_blank">Springerlink</a>        
      
                 
                       <a href="article/doi/10.1007/s12250-020-00228-z" target="_blank">ESM</a>
                    
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Severe fever with thrombocytopenia syndrome (SFTS), caused by SFTS virus (SFTSV) in the genus Banyangvirus of the family Phenuiviridae, is an emerging tick-borne viral zoonosis (Liu et al. 2014). Typical clinical symptoms of SFTS include fever, headache, thrombocytopenia, and leukocytopenia (Yu et al. 2011). Since SFTSV was identified in 2009, it has been found in more than 20 provinces in China and is closely related to strains from Japan and South Korea (Kim et al. 2013; Takahashi et al. 2014). In Northeastern China, the disease was discovered in Liaoning Province in 2010 (Wang et al. 2016), and the virus was detected in Haemaphysalis longicornis in Jilin Province in 2013 (Liu et al. 2016). Here we describe the first fatal case of SFTS in Jilin Province and conduct phylodynamic analysis of SFTSV.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00234-1" class="mainColor" target="_blank">Construction and Characterization of a Novel Recombinant Attenuated and Replication-Deficient Candidate Human Adenovirus Type 3 Vaccine: &#8220;Adenovirus Vaccine Within an Adenovirus Vector&#8221;</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Yuqian Yan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Shuping Jing</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Liqiang Feng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhiwei Zeng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Min Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Shan Zhao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Junxian Ou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wendong Lan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wenyi Guan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiaowei Wu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jianguo Wu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Donald Seto</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Qiwei Zhang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00234-1" target="_blank" class="list-doi">10.1007/s12250-020-00234-1</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>22 February 2020</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00234-1" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00234-1" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Human adenoviruses (HAdVs) are highly contagious and result in large number of acute respiratory disease (ARD) cases with severe morbidity and mortality. Human adenovirus type 3 (HAdV-3) is the most common type that causes ARD outbreaks in Asia, Europe, andtheAmericas.However, there is currently no vaccine approvedfor its general use.The hexon protein contains themain neutralizing epitopes, provoking strong and lasting immunogenicity. In this study, a novel recombinant and attenuated adenovirus vaccine candidate against HAdV-3 was constructed based on a commercially-available replication-defective HAdV-5 gene therapy and vaccine vector. The entire HAdV-3 hexon gene was integrated into the E1 region of the vector by homologous recombination using a bacterial system. Theresultantrecombinants expressing the HAdV-3 hexon protein were rescued in AD293 cells, identified and characterized by RT-PCR, Western blots, indirect immunofluorescence, and electron microscopy. This potential vaccine candidate had a similar replicative efficacy as the wild-type HAdV-3 strain. However, and importantly, the vaccine strain had been rendered replication-defective and was incapable of replication in A549 cells after more than twenty-generation passages in AD293 cells. This represents a significant safety feature. The mice immunized both intranasally and intramuscularly by this vaccine candidate raised significant neutralizing antibodies against HAdV-3. Therefore, this recombinant, attenuated, and safe adenovirus vaccine is a promising HAdV-3 vaccine candidate. The strategy of using a clinically approved and replication-defective HAdV-5 vector provides a novel approach to develop universal adenovirus vaccine candidates against all the other types of adenoviruses causing ARDs and perhaps other adenovirus-associated diseases.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00229-y" class="mainColor" target="_blank">Establishment of Murine Infection Models with Biological Clones of Dengue Viruses Derived from a Single Clinical Viral Isolate</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Zhihang Zheng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Min Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhihua Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xia Jin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jin Sun</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00229-y" target="_blank" class="list-doi">10.1007/s12250-020-00229-y</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>09 November 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00229-y" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00229-y" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Dengue virus (DENV) is a single-stranded RNA virus transmitted by mosquitoes in tropical and subtropical regions. It causes dengue fever, dengue hemorrhagic fever and dengue shock syndrome in patients. Each year, 390 million people are estimated to be infected by four serotypes of dengue virus, creating a great burden on global public health and local economy. So far, no antiviral drug is available for dengue disease, and the newly licensed vaccine is far from satisfactory. One large obstacle for dengue vaccine and drug development is the lack of suitable small animal models. Although some DENV infection models have been developed, only a small number of viral strains can infect immunodeficient mice. In this study, with biologically cloned viruses from a single clinical isolate, we have established two mouse models of DENV infection, one is severe lethal infection in immunocompromised mice, and the other resembles self-limited disease manifestations in Balb/c mice with transient blockage of type I IFN responses. This study not only offers new small animal models of dengue viral infection, but also provides new viral variants for further investigations on dengue viral pathogenesis.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00233-2" class="mainColor" target="_blank">Recombination of T4-like Phages and Its Activity against Pathogenic Escherichia coli in Planktonic and Biofilm Forms</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Min Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Donglin Shi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yanxiu Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yuyi Xiao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mianmian Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Liang Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hong Du</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei Zhang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00233-2" target="_blank" class="list-doi">10.1007/s12250-020-00233-2</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>09 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00233-2" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00233-2" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/056c4b9c-3e02-437c-b1e1-d38817ce108a.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">The increasing emergence of multi-drug resistant Escherichia coli (E. coli) has become a global concern, primarily due to the limitation of antimicrobial treatment options. Phage therapy has been considered as a promising alternative for treating infections caused by multi-drug resistant E. coli. However, the application of phages as a promising antimicrobial agent is limited by their narrow host range and specificity. In this research, a recombinant T4-like phage, named WGqlae, has been obtained by changing the receptor specificity determinant region of gene 37, using a homologous recombination platform of T4-like phages established by our laboratory previously. The engineered phage WGqlae can lyse four additional hosts, comparing to its parental phages WG01 and QL01. WGqlae showed similar characteristics, including thermo and pH stability, optimal multiplicity of infection and one-step growth curve, to the donor phage QL01. In addition, sequencing results showed that gene 37 of recombinant phage WGqlae had genetically stable even after 20 generations. In planktonic test, phage WGqlae had significant antimicrobial effects on E. coli DE192 and DE205B. The optical density at 600 nm (OD600) of E. coli in phage WGqlae treating group was significantly lower than that of the control group (P &lt; 0.01). Besides, phage WGqlae demonstrated an obvious inhibitory effect on the biofilm formation and the clearance of mature biofilms. Our study suggested that engineered phages may be promising candidates for future phage therapy applications against pathogenic E. coli in planktonic and biofilm forms.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00224-3" class="mainColor" target="_blank">Depletion but Activation of CD56dimCD16+ NK Cells in Acute Infection with Severe Fever with Thrombocytopenia Syndrome Virus</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Mengmeng Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yan Xiong</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mingyue Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wenjing Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jia Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yanfang Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Shue Xiong</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Congcong Zou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Boyun Liang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mengji Lu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Dongliang Yang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Cheng Peng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xin Zheng</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00224-3" target="_blank" class="list-doi">10.1007/s12250-020-00224-3</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>11 August 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00224-3" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00224-3" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/2e539f68-fb54-4ab3-b82e-bb879b9e22b2.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Severe fever with thrombocytopenia syndrome (SFTS) is an emerging infectious disease with high mortality (12%&#8211;30%). The mechanism by which the SFTS bunyavirus (SFTSV) causes severe illness remains unclear. To evaluate the phenotypic and functional characteristics of the NK cell subsets in SFTS patients, twenty-nine SFTS patients were sequentially sampled from admission until recovery. Phenotypic and functional characteristics of NK cell subsets in circulating blood were analysed via flow cytometry. Then, correlations between NK cell subset frequencies and the SFTS index (SFTSI) were evaluated in all SFTS patients (15 mild, 14 severe) upon admission. The frequencies of CD56dimCD16+ NK cells were greatly decreased in early SFTSV infection and were negatively correlated with disease severity. Additionally, higher Ki-67 and granzyme B expression and relatively lower NKG2A expression in CD56dimCD16+ NK cells were observed in acute infection. Moreover, the effector function of CD56dim NK cells was increased in the acute phase compared with the recovery phase in nine severe SFTS patients. Additionally, interleukin (IL)-15, interferon (IFN)-&#945;, IL-18 and IFN-&#947; secretion was markedly increased during early infection. Collectively, despite depletion of CD56dimCD16+ NK cells, activation and functional enhancement of CD56dimCD16+ NK cells were still observed, suggesting their involvement in defence against early SFTSV infection.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00226-1" class="mainColor" target="_blank">Myristoylation of EV71 VP4 is Essential for Infectivity and Interaction with Membrane Structure</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Jiaming Cao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Meng Qu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hongtao Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xuan Wan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Fang Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ali Hou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yan Zhou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Bo Sun</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Linjun Cai</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Weiheng Su</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Chunlai Jiang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00226-1" target="_blank" class="list-doi">10.1007/s12250-020-00226-1</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>12 December 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00226-1" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00226-1" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/871aa9b9-619f-40ba-828a-f8d06555c2ae.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">The Enterovirus 71 (EV71) VP4 is co-translationally linked to myristic acid at its amino-terminal glycine residue. However, the role of this myristoylation in the EV71 life cycle remains largely unknown. To investigate this issue, we developed a myristoylation-deficient virus and reporter (luciferase) pseudovirus with a Gly-to-Ala mutation (G2A) on EV71 VP4. When transfecting the EV71-G2A genome encoding plasmid in cells, the loss of myristoylation on VP4 did not affect the expression of viral proteins and the virus morphology, however, it did significantly influence viral infectivity. Further, in myristoylation-deficient reporter pseudovirus-infected cells, the luciferase activity and viral genome RNA decreased significantly as compared to that of wild type virus; however, cytopathic effect and viral capsid proteins were not detected in myristoylation-deficient virus-infected cells. Also, although myristoylation-deficient viral RNA and proteins were detected in the second blind passage of infection, they were much fewer in number compared to that of the wild type virus. The replication of genomic RNA and negative-strand viral RNA were both blocked in myristoylation-deficient viruses, suggesting that myristoylation affects viral genome RNA release from capsid to cytoplasm. Besides, loss of myristoylation on VP4 altered the distribution of VP4-green fluorescent protein protein, which disappeared from the membrane structure fraction. Finally, a liposome leakage assay showed that EV71 myristoylation mediates the permeability of the model membrane. Hence, the amino-terminal myristoylation of VP4 is pivotal to EV71 infection and capsidmembrane structure interaction. This study provides novel molecular mechanisms regarding EV71 infection and potential molecular targets for antiviral drug design.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00225-2" class="mainColor" target="_blank">Molecular Epidemiology and Vaccine Compatibility Analysis of Seasonal Influenza Viruses in Wuhan, 2016&#8211;2019</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Liang-Jun Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing-Jing Guo</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei-Wei Guo</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  E-Xiang Shen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xin Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Kai-Ji Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jie Yan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mang Shi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yi-Rong Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei Hou</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00225-2" target="_blank" class="list-doi">10.1007/s12250-020-00225-2</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>15 December 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00225-2" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00225-2" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/79e030da-4992-4d9a-a067-2c5319196faa.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Influenza viruses (FLUV) cause high morbidity and mortality annually in the world and pose a serious threat to the public health. Wuhan, as an important transportation hub in China, has a dense population and suitable climate, which also lays a major hidden danger for the outbreak of influenza. To survey and characterize the seasonal FLUV in Wuhan during 2016&#8211;2019, we collected 44,738 throat swabs, among which 15.5% were influenza A (FLUAV) positive, 6.1% influenza B (FLUBV) and 0.3% co-infection. By monitoring FLUV in each month from June 2016 to May 2019, different with the previously seasonality pattern, only a single influenza peak was appeared in winter of 2017&#8211;2018 and 2018&#8211;2019, respectively. These data indicated that the complex circulation pattern of seasonal influenza in Wuhan. In addition, we found the age group was skewed towards 5&#8211;14 years group whose activity were mostly school based, which suggested school may be an important place for influenza outbreaks. Meanwhile, phylogenic analysis revealed that two subtypes (subclade 3C.2a2 and 3C.2a1b) of A(H3N2) were circulating in Wuhan and there was an obvious transition in 2018 because the two subclades were detected simultaneously. Furthermore, by estimating the vaccine effectiveness, we found that the vaccine strain of FLUAV didn&#8217;t seem to match very well the current epidemic strain, especially A(H3N2). Hence, more accurate prediction of seasonal outbreak is essential for vaccine design. Taken together, our results provided the current information about seasonal FLUV in Wuhan which form the basis for vaccine updating.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00193-7" class="mainColor" target="_blank">Phylogeography of Highly Pathogenic H5 Avian Influenza Viruses in China</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Xiaowen Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xueying Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Bing Xu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00193-7" target="_blank" class="list-doi">10.1007/s12250-020-00193-7</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>26 May 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00193-7" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00193-7" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/5dc1d719-b169-4051-9ea0-d72127aa1069.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">The spread of H5 highly pathogenic avian influenza viruses poses serious threats to the poultry industry, wild bird ecology and human health. Circulation of H5 viruses between poultry and wild birds is a significant public health threat in China. Thus, viral migration networks in this region need to be urgently studied. Here, we conducted molecular genetic analyses of the hemagglutinin genes of H5 highly pathogenic avian influenza viruses in multiple hosts from 2000 to 2018 in China. Our aim was to clarify the roles of different hosts in the evolution of H5 viruses. We used a flexible Bayesian statistical framework to simulate viral space diffusion and continuous-time Markov chains to infer the dynamic evolutionary process of spatiotemporal dissemination. Bayesian phylogeographic analysis of H5 viruses showed for the first time that H5 viruses in poultry and wild birds were present in Guangdong Province. Furthermore, Guangdong, Jiangsu, Shanghai and Hunan acted as the epicenters for the spread of various H5 subtypes viruses in poultry, and Henan, Shanghai, Hong Kong and Inner Mongolia acted as epicenters for the spread of various H5 subtypes viruses in wild birds. Thus, H5 viruses exhibited distinct evolutionary dynamics in poultry and wild birds. Our findings extend our understanding of the transmission and spread of highly pathogenic H5 avian influenza viruses in China.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00222-5" class="mainColor" target="_blank">A Comprehensive Review on Human Aichi Virus</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Enrique Rivadulla</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jes&#250;s L. Romalde</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00222-5" target="_blank" class="list-doi">10.1007/s12250-020-00222-5</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>21 September 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00222-5" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00222-5" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Although norovirus, rotavirus, adenovirus and Astrovirus are considered the most important viral agents transmitted by food and water, in recent years other viruses, such as Aichi virus (AiV), have emerged as responsible for gastroenteritis outbreaks associated with different foods. AiV belongs to the genus Kobuvirus of the family Picornaviridae. It is a virus with icosahedral morphology that presents a single stranded RNA genome with positive sense (8280 nucleotides) and a poly (A) chain. AiV was first detected from clinical samples and in recent years has been involved in acute gastroenteritis outbreaks from different world regions. Furthermore, several studies conducted in Japan, Germany, France, Tunisia and Spain showed a high prevalence of AiV antibodies in adults (between 80% and 99%), which is indicative of a large exposure to this virus. The aim of this review is to bring together all the discovered information about the emerging pathogen human Aichi virus (AiV), discussing the possibles routes of transmission, new detection techniques and future research. Although AiV is responsible for a low percentage of gastroenteritis outbreaks, the high seroprevalence shown by human populations indicates an evident role as an enteric agent. The low percentage of AiV detection could be explained by the fact that the pathogen is more associated to subclinical infections. Further studies will be needed to clarify the real impact of AiV in human health and its importance as a causative gastroenteritis agent worldwide.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00216-3" class="mainColor" target="_blank">IP10, KC and M-CSF is Remarkably Increased in the Brains from the Various Strains of Experimental Mice Infected with Different Scrapie Agents</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Jia Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Cao Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Chao Hu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lian Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ying Xia</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lin Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei Yang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hai-Yan Wu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei Zhou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Kang Xiao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Qi Shi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yuezhang Wu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhi-Bao Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiao-Ping Dong</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00216-3" target="_blank" class="list-doi">10.1007/s12250-020-00216-3</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>17 September 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00216-3" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00216-3" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/7036db20-6818-4aeb-8317-d3bb5ebe3f23.docx" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Activation of inflammatory cells and upregulations of a number of cytokines in the central nervous system (CNS) of patients with prion diseases are frequently observed. To evaluate the potential changes of some brain cytokines that were rarely addressed during prion infection, the levels of 17 different cytokines in the brain homogenates of mice infected with different scrapie mouse-adapted agents were firstly screened with Luminex assay. Significant upregulations of interferon gamma-induced protein 10 (IP10), keratinocyte chemoattractant (KC) and macrophage colony stimulating factor (M-CSF) were frequently detected in the brain lysates of many strains of scrapie infected mice. The upregulations of those three cytokines in the brains of scrapie infected mice were further validated by the individual specific ELISA and immunohistochemical assay. Increased specific mRNAs of IP10, M-CSF and KC in the brains of scrapie infected mice were also detected by the individual specific qRT-PCRs and IP10-specific digital PCR. Dynamic analyses of the brain samples collected at different time points post infection revealed the time-dependent increases of those three cytokines, particularly IP10 during the incubation period of scrapie infection. In addition, we also found that the levels of IP10 in cerebral spinal fluid (CSF) of 45 sporadic Creutzfeldt&#8211;Jakob disease (sCJD) patients were slightly but significantly higher than those of the cases who were excluded the diagnosis of prion diseases. These data give us a better understanding of inflammatory reaction during prion infection and progression of prion disease.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00213-6" class="mainColor" target="_blank">Cutaneous Dengue Virus Inoculation Triggers Strong B Cell Reactions but Contrastingly Poor T Cell Responses</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Edith Marcial-Ju&#225;rez</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Julio Garc&#237;a-Cordero</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ra&#250;l Antonio Maqueda-Alfaro</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Rafael Eduardo Saucedo-L&#243;pez</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Luvia Enid S&#225;nchez-Torres</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Leticia Cedillo-Barr&#243;n</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Leopoldo Flores-Romo</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00213-6" target="_blank" class="list-doi">10.1007/s12250-020-00213-6</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>16 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00213-6" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00213-6" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Dengue is a global health problem without current specific treatment nor safe vaccines available. While severe dengue is related to pre-existing non-neutralizing dengue virus (DENV) antibodies, the role of T cells in protection or pathology is unclear. Using cutaneous DENV infection in immunocompetent mice we previously showed the generation of PNA+ germinal centers (GCs), now we assessed the activation and proliferation of B and T cells in draining lymph nodes (DLNs). We found a drastic remodelling of DLN compartments from 7 to 14 days post-infection (dpi) with greatly enlarged B cell follicles, occupying almost half of the DLN area compared to ~24% in na&#239;ve conditions. Enormous clusters of proliferating (Ki-67+) cells inside B follicles were found 14 dpi, representing ~33% of B cells in DLNs but only ~2% in noninfected mice. Inside GCs, we noticed an important recruitment of tingle body macrophages removing apoptotic cells. In contrast, the percentage of paracortex area and total T cells decreased by 14&#8211;16 dpi, compared to controls. Scattered randomly distributed Ki-67+ T cells were found, similar to non-infected mice. CD69 expression by CD4+ and CD8+ T cells was minor, while it was remarkable in B cells, representing 1764.7% of change from basal levels 3 dpi. The apparent lack of T cell responses cannot be attributed to apoptosis since no significant differences were observed compared to noninfected mice. This study shows massive B cell activation and proliferation in DLNs upon DENV infection. In contrast, we found very poor, almost absent CD4+ and CD8+ T cell responses.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00220-7" class="mainColor" target="_blank">Effect of Peroxisome Proliferator-Activated Receptor-&#947; Coactivator-1 Alpha Variants on Spontaneous Clearance and Fibrosis Progression during Hepatitis C Virus Infection in Moroccan Patients</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Raouia ElFihry</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mohcine Elmessaoudi-Idrissi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Fatima-Zahra Jadid</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Imane Zaidane</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hajar Chihab</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mohamed Tahiri</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mostafa Kabine</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wafaa Badre</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Isabelle Chemin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Agnes Marchio</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Pascal Pineau</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Sayeh Ezzikouri</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Soumaya Benjelloun</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00220-7" target="_blank" class="list-doi">10.1007/s12250-020-00220-7</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>04 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00220-7" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00220-7" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Hepatitis C virus (HCV) is still one of the main causes of liver disease worldwide. Metabolic disorders, including nonalcoholic fatty liver disease (NAFLD), induced by HCV have been shown to accelerate the progression of fibrosis to cirrhosis and to increase the risk of hepatocellular carcinoma. An optimal peroxisome proliferator-activated receptor gamma coactivator 1-alpha (PPARGC1A) activity is crucial to prevent NAFLD installation. The present study aims to investigate the associations between two common PPARGC1A polymorphisms (rs8192678 and rs12640088) and the outcomes of HCV infection in a North African context. A series of 592 consecutive Moroccan subjects, including 292 patients with chronic hepatitis C (CHC), 100 resolvers and 200 healthy controls were genotyped using a TaqMan allelic discrimination assay. PPARGC1A variations at rs8192678 and rs12640088 were not associated with spontaneous clearance of HCV infection (adjusted ORs = 0.76 and 0.79 respectively, P &gt;0.05, for both). Furthermore, multivariable logistic regression analysis showed that both SNPs were not associated with fibrosis progression (OR = 0.71; 95% CI 0.20&#8211;2.49; P = 0.739; OR = 1.28; 95% CI 0.25&#8211;6.54; P = 0.512, respectively). We conclude that, in the genetic context of South Mediterranean patients, rs8192678 and rs12640088 polymorphisms of PPARGC1A are neither associated with spontaneous clearance nor with disease progression in individuals infected with HCV.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00200-x" class="mainColor" target="_blank">New Gene Variants Associated with the Risk of Chronic HBV Infection</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Mengjie Fan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Sa Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Tengyan Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hong Pan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hankui Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Huifang Xu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Daria V. Zhernakova</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Stephen J. O'Brien</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhenru Feng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Le Chang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Erhei Dai</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jianhua Lu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hongli Xi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yanyan Yu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jianguo Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Binbin Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zheng Zeng</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00200-x" target="_blank" class="list-doi">10.1007/s12250-020-00200-x</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>05 March 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00200-x" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00200-x" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/61ad234e-e17a-4665-8caa-66c3a9c633d0.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Some patients with chronic hepatitis B virus (HBV) infection failed to clear HBV, even persistently continue to produce antibodies to HBV. Here we performed a two stage genome wide association study in a cohort of Chinese patients designed to discover single nucleotide variants that associate with HBV infection and clearance of HBV. The first stage involved genome wide exome sequencing of 101 cases (HBsAg plus anti-HBs positive) compared with 102 control patients (antiHBs positive, HBsAg negative). Over 80% of individual sequences displayed 20&#215;sequence coverage. Adapters, uncertain bases &gt;10% or low-quality base calls (&gt;50%) were filtered and compared to the human reference genome hg19. In the second stage, 579 chronic HBV infected cases and 439 HBV clearance controls were sequenced with selected genes from the first stage. Although there were no significant associated gene variants in the first stage, two significant gene associations were discovered when the two stages were assessed in a combined analysis. One association showed rs506121-"T" allele [within the dedicator of cytokinesis 8 (DOCK8) gene] was higher in chronic HBV infection group than that in clearance group (P = 0.002, OR = 0.77, 95% CI [0.65, 0.91]). The second association involved rs2071676&#8212;A allele within the Carbonic anhydrase (CA9) gene that was significantly elevated in chronic HBV infection group compared to the clearance group (P = 0.0003, OR = 1.35, 95% CI [1.15, 1.58]). Upon replication these gene associations would suggest the influence of DOCK8 and CA9 as potential risk genetic factors in the persistence of HBV infection.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00199-1" class="mainColor" target="_blank">Chimeric Newcastle Disease Virus-like Particles Containing DC-Binding Peptide-Fused Haemagglutinin Protect Chickens from Virulent Newcastle Disease Virus and H9N2 Avian Influenza Virus Challenge</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Xiaohong Xu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing Qian</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lingsong Qin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jindou Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Cong Xue</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jiaxin Ding</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Weiqi Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei Ding</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Renfu Yin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ningyi Jin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhuang Ding</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00199-1" target="_blank" class="list-doi">10.1007/s12250-020-00199-1</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>03 February 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00199-1" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00199-1" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Newcastle disease virus (NDV) and H9N2 subtype Avian influenza virus (AIV) are two notorious avian respiratory pathogens that cause great losses in the poultry industry. Current inactivated commercial vaccines against NDV and AIV have the disadvantages of inadequate mucosal responses, while an attenuated live vaccine bears the risk of mutation. Dendritic cell (DC) targeting strategies are attractive for their potent mucosal and adaptive immune-stimulating ability against respiratory pathogens. In this study, DC-binding peptide (DCpep)-decorated chimeric virus-like particles (cVLPs), containing NDV haemagglutinin&#8211;neuraminidase (HN) and AIV haemagglutinin (HA), were developed as a DC-targeting mucosal vaccine candidate. DCpep-decorated cVLPs activated DCs in vitro, and induced potent immune stimulation in chickens, with enhanced secretory immunoglobulin A (sIgA) secretion and splenic T cell differentiation. 40 &#956;g cVLPs can provide full protection against the challenge with homologous, heterologous NDV strains, and AIV H9N2. In addition, DCpep-decorated cVLPs could induce a better immune response when administered intranasally than intramuscularly, as indicated by robust sIgA secretion and a reduced virus shedding period. Taken together, this chimeric VLPs are a promising vaccine candidate to control NDV and AIV H9N2 and a useful platform bearing multivalent antigens.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00217-2" class="mainColor" target="_blank">Potential m6A and m5C Methylations within the Genome of A Chinese African Swine Fever Virus Strain</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Lijia Jia</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jianjun Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Haizhou Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wenhui Fan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Depeng Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Di Liu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00217-2" target="_blank" class="list-doi">10.1007/s12250-020-00217-2</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>05 January 2020</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00217-2" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00217-2" target="_blank">Springerlink</a>        
      
                 
                       <a href="article/doi/10.1007/s12250-020-00217-2" target="_blank">ESM</a>
                    
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">It has been more than 1 year since China reported the first case of African swine fever (ASF) infection in August 2018, and the epidemic situation remains severe (China News Service 2019). According to reports from the Ministry of Agriculture and Rural Affairs, China has reported 160 cases of ASF, which resulted in nearly 1.2 million pigs being killed, as of November 21, 2019 (China News Service 2019). ASF is an acute febrile, hemorrhagic and fulminating infectious disease, and would reach 100% case fatality rate to pigs (Gallardo et al. 2015). The causative pathogen, African swine fever virus (ASFV), is a doublestranded DNA virus with a genome of 170&#8211;193 kb belonging to the Asfarviridae family (Galindo and Alonso 2017; Gallardo et al. 2015). A recent study has revealed that ASFV maintains a core genome of 102 ORFs and has 168 dispensable genes (Wang et al. 2019). Thus, the complexed genomic features of ASFV require more attentions. By using the next generation sequencing (NGS) and the single molecule real-time sequencing (SMRT-seq), a couple of Chinese ASFV genomes have been uncovered (Bao et al. 2019; Wen et al. 2019; Jia et al. 2019). Compared to NGS, SMRT-seq has the advantage of long read length and can generate sequencing data containing the original single base modification information, which can be identified through the state-of-art bioinformatic procedures (Senol Cali et al. 2019; Simpson et al. 2017). DNA methylation is a chemical modification common in animal and plant genomes. It refers to the catalytic transfer of methyl groups on active methyl compounds (such as s-adenosine methionine) to other compounds under the catalysis of DNA methyltransferase (DNMT), mainly forming 5-methylcytosine (5-mC), 6-methyladenine (6-mA), 5-hydroxymethylcytosine (5-hmC), etc. DNA methylation, which triggers the epigenetic regulatory mechanism, has been proved to play important roles in gene expression and regulation, embryonic development, and disease-related aspects (Gouil and Keniry 2019). Whether ASFV genome has DNA methylation and epigenetic regulation is to be discerned.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00211-8" class="mainColor" target="_blank">Development of an MCA-Based Real Time RT-qPCR Assay for the Simultaneous Detection and Differentiation of Duck Hepatitis A Virus Types 1 and 3</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Chunchun Meng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yunxiu Huang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zaib Ur Rehman</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wen Hu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Chuanfeng Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ruiying Liang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zongyan Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Kaijie Song</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Tianchao Wei</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Guangqing Liu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00211-8" target="_blank" class="list-doi">10.1007/s12250-020-00211-8</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>16 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00211-8" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00211-8" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Duck virus hepatitis (DVH) is a significant concern in the duck industry as the disease causes a highly contagious infection in young ducklings that is often associated with liver necrosis, hemorrhage, and high mortality (Yugo et al. 2016). Duck hepatitis virus (DHV) was first described in 1949 on Long Island in the United States. Subsequent, outbreaks have been reported in England, Canada, Germany, Japan and elsewhere (Toth 1969). DHV is associated with at least two RNA viruses, duck hepatitis A virus (DHAV) and duck astrovirus (DAstV); however, no antigenic relationships have been identified between these two viruses (Yugo et al. 2016). DHAV is the primary causative agent of DVH. As the only member of the genus Avihepatovirus, in the Picornaviridae family, DHAV has a linear, single-stranded positive-sense RNA genome. The genomic organization of DHAV is analogous to that of other picornaviruses with one large open reading frame (ORF) that encodes a polyprotein precursor, that is preceded by a 50-untranslated-terminal-region (UTR) and followed by 30- UTR (Tseng et al. 2007). Based on systematic phylogenetic analyses and neutralization assays, DHAVs have been classified into three serotypes: the classical serotype 1 (DHAV-1) (Kim et al. 2006; Ding and Zhang 2007; Tseng et al. 2007), the second serotype that has only been reported in Taiwan Province of China (DHAV-2) (Tseng and Tsai 2007), and the third serotype that was first reported in South Korea (DHAV-3) (Kim et al. 2007). DHAV-3 also accounts for an increasing proportion of DHV pathogens in China (Liu et al. 2011; Zhang et al. 2017; Wen et al. 2018), South Korea (Cha et al. 2013; Soliman et al. 2015) and Vietnam (Doan et al. 2016).</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00210-9" class="mainColor" target="_blank">Characterization of the First Genome of Porcine mastadenovirus B (HNU1 Strain) and Implications on Its Lymphoid and Special Origin</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Shu-Jing Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Qiong Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ting-Ting Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Si-Hua Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jin-Yan Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Li-Jun Wu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ye Qiu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xing-Yi Ge</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00210-9" target="_blank" class="list-doi">10.1007/s12250-020-00210-9</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>05 December 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00210-9" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00210-9" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/b3e90d52-acec-42cc-b42f-25802244b1c2.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Porcine adenoviruses (PAdVs) are classified into three species, PAdV-A, PAdV-B, and PAdV-C. The genomes of PAdV-A and PAdV-C have been well characterized. However, the genome of PAdV-B has never been completely sequenced, and the epidemiology of PAdV-B remains unclear. In our study, we have identified a novel strain of PAdV-B, named PAdV-BHNU1, in porcine samples collected in China by viral metagenomic assay and general PCR. The genome of PAdV-BHNU1 is 31,743 bp in length and highly similar to that of California sea lion adenovirus 1 (C. sea lion AdV-1), which contains typical mastadenoviral structures and some unique regions at the carboxy-terminal end. Especially, PAdV-BHNU1 harbors a dUTPase coding region not clustering with other mastadenoviruses except for C. sea lion AdV-1 and a fiber coding region homologous with galectin 4 and 9 of animals. However, the variance of GC contents between PAdV-BHNU1 (55%) and C. sea lion AdV-1 (36%) indicates their differential evolutionary paths. Further epidemiologic study revealed a high positive rate (51.7%) of PAdV-B-HNU1 in porcine lymph samples, but low positive rates of 10.2% and 16.1% in oral swabs and rectal swabs, respectively. In conclusion, this study characterized a novel representative genome of a lymphotropic PAdV-B with unique evolutionary origin, which contributes to the taxonomical and pathogenic studies of PAdVs.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00209-2" class="mainColor" target="_blank">Prion Protein Expression is Correlated with Glioma Grades</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Qiaoli Luo</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yisong Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Dongying Fan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Shijie Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Peigang Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing An</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00209-2" target="_blank" class="list-doi">10.1007/s12250-020-00209-2</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>08 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00209-2" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00209-2" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Glioma is the most common primary central nervous system (CNS) tumors in human. Gliomas are classified into low-grade glioma (LGG, grades I&#8211;II) and high-grade glioma (HGG, grades III&#8211;IV) according to the World Health Organization classification guidelines (Louis et al. 2007). Comparing to the LGG, the HGG becomes more aggressive and has a poorer median overall survival. At present, glioma is mainly treated by radiotherapy, chemotherapy and surgical resection, but the effect is not ideal, especially for the glioblastoma (GBM), which is the most malignant type of gliomas and is resistant to current therapy protocols. There are several theories about the pathogenesis of glioma, such as radiation (Ohgaki and Kleihues 2005), genetics (Farrell and Plotkin 2007; Goodenberger and Jenkins 2012) and viruses (Lawler. 2015; Xing et al. 2016), while its etiology remains obscure. Therefore, it is urgent to investigate the causes and to identify the promising molecules that target the glioma.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00201-w" class="mainColor" target="_blank">Re-emergence of Highly Pathogenic Avian Influenza H5N1 in Nigeria, 2014&#8211;2016: Role of Social Network and Value Chain Forces in Interstate Transmission</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Daniel Oladimeji Oluwayelu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Clement Adebajo Meseko</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Adekunle Bamidele Ayinmode</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Adebowale Idris Adebiyi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mike Aneshimi Lawani</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Florence Omonele Kakulu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00201-w" target="_blank" class="list-doi">10.1007/s12250-020-00201-w</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>07 August 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00201-w" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00201-w" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Since 2004, high pathogenic avian influenza (HPAI) due to H5N1 virus among others has caused a major veterinary health crisis, resulting in the loss of millions of poultry through death or culling in several countries in Asia, the Middle East, Europe, Africa and North America (Alexander 2007; Lee et al. 2016). The first African outbreak was reported from Nigeria in 2006 in domestic poultry (Joannis et al. 2006) and persisted till 2008 (Fusaro et al. 2010). These outbreaks negatively affected animal and public health as well as the economy, and were caused by viruses belonging to genetic clades 2.2, 2.2.1, 2.2.2 and 2.3.2.1c (Aiki-Raji et al. 2008; Fusaro et al. 2010; Monne et al. 2015).</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00197-3" class="mainColor" target="_blank">Comparison of Viromes in Ticks from Different Domestic Animals in China</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Tingting Zhao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Haiyan Gong</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiaojuan Shen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wen Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Tongling Shan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiangqian Yu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Seong Jin Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Li Cui</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00197-3" target="_blank" class="list-doi">10.1007/s12250-020-00197-3</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>08 May 2019</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>10 March 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>10 March 2020</span>
	            	
            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00197-3" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00197-3" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Ticks are involved in the transmission of various arboviruses and some tick-borne viruses pose significant threats to the health of humans or livestock. This study aimed to investigate the geographical distribution of tick species and tickassociated viruses in central and eastern China. Total 573 ticks from domestic animals including dogs, sheep and cattle were collected in 2017. Two genera of ticks were identified including Rhipicephalus and Haemaphysalis. Sequencing was performed on Miseq Illumina platform to characterize the tick viromes from the four different sampling locations. Following trimming, 13,640 reads were obtained and annotated to 19 virus families. From these sequences, above 37.74% of the viral reads were related to the RNA viruses. Virome comparison study revealed that the tick viral diversity was considerably different in the two identified tick genera. The viral diversity of R. microplus was significantly different from that of other Rhipicephalus species. On the other hand, substantial overlap in viral species was observed between the same genera. In addition, we found no evidence that the natural host played a major role in shaping virus diversity based on the comparison of their viromes. Rather, the geographic location seems to significantly influence the viral families. Phylogenetic study indicated that the novel negative-sense RNA viruses identified in this study was closely related to Bole tick virus 1 and 3 viruses. In conclusion, the present study provides a baseline for comparing viruses detected in ticks, according to species, natural hosts, and geographic locations.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00202-9" class="mainColor" target="_blank">A Mother-to-Child Transmission Study in Nigeria: The Impact of Maternal HIV Infection and HAART on Plasma Immunoglobulins, Cytokine Profiles and Infant Outcome</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Chinwe O. Ewenighi-Amankwah</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Charles Chinedum Onyenekwe</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ogochukwu Udemba</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Patience Muogbo</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lijun Rong</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00202-9" target="_blank" class="list-doi">10.1007/s12250-020-00202-9</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>09 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00202-9" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00202-9" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Prevention of mother-to-child transmission (PMTCT) of HIV with highly active antiretroviral therapy (HARRT) allows the HIV+ pregnant mothers to have vaginal delivery and breastfeed. Here we investigated the maternal plasma immunoglobulin, cytokine secretion and the outcome of the exposed infants among the HIV+ HAART treated pregnant women in Nigeria. In this study, different plasma immunoglobulins and cytokines were measured in the HIV+ HAART treated pregnant mothers. Pooled culture supernatants of B and T lymphocytes showed lower levels of IFN-&#947;, IL-10 and IL-4. There were lower IFN-&#947; and IL-10 secretions at 1st trimester; however, IL-10 continued to be lower throughout 2nd and 3rd trimesters. TNF-&#945; secretion significantly decreased as pregnancy progressed to term. There were high plasma IgG and low IgM in the HIV+ HAART treated pregnant women. Plasma IgG was high during 1st and 3rd trimesters. After one year of follow up, all the exposed children were seronegative for HIV-1 and HIV-2. Vaginal delivery and breastfeeding among HIV+ HAART treated mothers have shown to be safe. The use of HAART by the infected mothers and the use of septrin and niverapin by the exposed infants prevented mother to-child transmission of HIV.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00208-3" class="mainColor" target="_blank">The Restrictome of Flaviviruses</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Lionel Berthoux</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00208-3" target="_blank" class="list-doi">10.1007/s12250-020-00208-3</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>30 November 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00208-3" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00208-3" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Flaviviruses are a genus of mostly arthropod-borne RNA viruses that cause a range of pathologies in humans. Basic knowledge on flaviviruses is rapidly expanding, partly due to their status as frequent emerging or re-emerging pathogens. Flaviviruses include the dengue, Zika, West Nile, tick-borne encephalitis and yellow fever viruses (DENV, ZIKV, WNV, TBEV and YFV, respectively). As is the case with other families of viruses, the success of productive infection of human cells by flaviviruses depends in part on the antiviral activity of a heterogeneous group of cellular antiviral proteins called restriction factors. Restriction factors are the effector proteins of the cell-autonomous innate response against viruses, an immune pathway that also includes virus sensors as well as intracellular and extracellular signal mediators such as type I interferons (IFN-I). In this review, I summarize recent progress toward the identification and characterization of flavivirus restriction factors. In particular, I focus on IFI6, Schlafen 11, FMRP, OAS-RNase L, RyDEN, members of the TRIM family of proteins (TRIM5&#945;, TRIM19, TRIM56, TRIM69 and TRIM79&#945;) and a new mechanism of action proposed for viperin. Recent and future studies on this topic will lead to a more complete picture of the flavivirus restrictome, defined as the ensemble of cellular factors with demonstrated anti-flaviviral activity.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00198-2" class="mainColor" target="_blank">The Establishment of Infectious Clone and Single Round Infectious Particles for Coxsackievirus A10</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Min Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jingjing Yan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Liuyao Zhu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Meng Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lizhen Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Rui Yu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ming Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jingna Xun</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yuling Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhigang Yi</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Shuye Zhang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00198-2" target="_blank" class="list-doi">10.1007/s12250-020-00198-2</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>27 September 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00198-2" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00198-2" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/0e82bd0b-1930-430f-af73-22696da6a250.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Coxsackievirus A10 (CVA10) is one of the major etiological agents of hand, foot, and mouth disease. There are no vaccine and antiviral drugs for controlling CVA10 infection. Reverse genetic tools for CVA10 will benefit its mechanistic study and development of vaccines and antivirals. Here, two infectious clones for the prototype and a Myc-tagged CVA10 were constructed. Viable CVA10 viruses were harvested by transfecting the viral mRNA into human rhabdomyosarcoma (RD) cells. Rescued CVA10 was further confirmed by next generation sequencing and characterized experimentally. We also constructed the vectors for CVA10 subgenomic replicon with luciferase reporter and viral capsid with EGFP reporter, respectively. Co-transfection of the viral replicon RNA and capsid expresser in human embryonic kidney 293T (HEK293T) cells led to the production of single round infectious particles (SRIPs). Based on CVA10 replicon RNA, SRIPs with either the enterovirus A71 (EVA71) capsid or the CVA10 capsid were generated. Infection by EVA71 SRIPs required SCARB2, while CVA10 SRIPs did not. Finally, we showed great improvement of the replicon activity and SRIPs production by insertion of a cis-active hammerhead ribozyme (HHRib) before the 50-untranslated region (UTR). In summary, reverse genetic tools for prototype strain of CVA10, including both the infectious clone and the SRIPs system, were successfully established. These tools will facilitate the basic and translational study of CVA10.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00194-6" class="mainColor" target="_blank">Conservative Evolution of Hepatitis B Virus Precore and Core Gene During Immune Tolerant Phase in Intrafamilial Transmission</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Yuqian Luo</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Le Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yimin Dai</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yali Hu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Biyun Xu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yi-Hua Zhou</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00194-6" target="_blank" class="list-doi">10.1007/s12250-020-00194-6</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>02 August 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00194-6" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00194-6" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/13517130-d98e-4bb1-87a8-dffc1171a16a.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Hepatitis B virus (HBV) is characterized with high mutations, which is attributed to the lack of proof-reading of the viral reverse transcriptase and host immune pressure. In this study, 31 HBV chronic carriers from 14 families were enrolled to investigate the evolution of the same original HBV sources in different hosts. Sequences of pre-C and C (pre-C/C) genes were analyzed in eight pairs of HBV-infected mothers with longitudinal sera (at an interval of 6.0&#8211;7.2 years) and their children (5.5&#8211;6.7 years old), and in 15 adults (21&#8211;78 years old) from six families with known intrafamilial HBV infection. The pre-C/C sequences had almost no change in eight mothers during 6.0&#8211;7.2 years and their children who were in immune tolerant phase. The pre-C/C sequences from the 15 adults of six families, mostly in the immune-clearance phase or the low replicative phase, showed various diversified mutations between individuals from each family. Compared to a reference stain (GQ205441) isolated nearby, the pre-C/C in individuals in immune tolerant phase showed 98.56%&#8211;99.52% homology at nucleotide level and 99.5%&#8211;100% homology at amino acid level. In contrast, multiple mutations were developed in the immune-clearance phase or the low replicative phase, affecting immune epitopes in core gene and G1896 in pre-C gene. The results indicate that the evolution of new HBV variants is not mainly resulted from the spontaneous error rate of viral reverse transcription, but from the host immune pressure.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00195-5" class="mainColor" target="_blank">Sero-Epidemiological Survey of Crimean-Congo Hemorrhagic Fever among the Human Population of the Punjab Province in Pakistan</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Muhammad Furqan Shahid</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Muhammad Zubair Shabbir</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Kamran Ashraf</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Muzaffar Ali</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Saima Yaqub</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Aziz Ul-Rahman</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Nageen Sardar</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Nadia Mukhtar</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zarfishan Tahir</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Tahir Yaqub</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00195-5" target="_blank" class="list-doi">10.1007/s12250-020-00195-5</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>18 April 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00195-5" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00195-5" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/06c7cc30-f694-46ed-b663-758dbba92776.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Crimean-Congo hemorrhagic fever (CCHF), caused by the CCHF virus (CCHFV), is a tick-borne zoonotic infection characterized by myalgia, high-grade fever (&gt; 38 &#8451;), headache, nausea, bleeding from the body cavities, and in 10%&#8211;50% of cases, results in death (Swanepoel et al. 1989). As CCHFV belongs to the Nairoviridae family, the virus can be transmitted to humans through the bite of infected ticks or by contact with the tissues or blood of infected animals (Bente et al. 2013).</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-020-00196-4" class="mainColor" target="_blank">Structure of the HRV-C 3C-Rupintrivir Complex Provides New Insights for Inhibitor Design</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Shuai Yuan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Kaiyue Fan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Zhonghao Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yao Sun</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Hai Hou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ling Zhu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-020-00196-4" target="_blank" class="list-doi">10.1007/s12250-020-00196-4</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>21 August 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-020-00196-4" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-020-00196-4" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Human rhinoviruses (HRVs) are the predominant infectious agents for the common cold worldwide. The HRV-C species cause severe illnesses in children and are closely related to acute exacerbations of asthma. 3C protease, a highly conserved enzyme, cleaves the viral polyprotein during replication and assists the virus in escaping the host immune system. These key roles make 3C protease an important drug target. A few structures of 3Cs complexed with an irreversible inhibitor rupintrivir have been determined. These structures shed light on the determinants of drug specificity. Here we describe the structures of HRV-C15 3C in free and inhibitor-bound forms. The volume-decreased S1&#8217; subsite and half-closed S2 subsite, which were thought to be unique features of enterovirus A 3C proteases, appear in the HRV-C 3C protease. Rupintrivir assumes an "intermediate" conformation in the complex, which might open up additional avenues for the design of potent antiviral inhibitors. Analysis of the features of the three-dimensional structures and the amino acid sequences of 3C proteases suggest new applications for existing drugs.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00191-4" class="mainColor" target="_blank">Hepatitis C Virus Infection Caused by Infrequent Exposure in China Should Be of Concern</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Xin-Cheng Qin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Li-Hua Zhong</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Li-Ying Zhu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Alexander Plyusnin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yong-Zhen Zhang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00191-4" target="_blank" class="list-doi">10.1007/s12250-019-00191-4</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>26 April 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00191-4" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00191-4" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/6b0fff1a-e2d8-476a-af08-891bb280f706.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Hepatitis C Virus (HCV) infection is a huge public health problem globally, because it can lead to adverse long-term clinical outcomes. In China, the population has experienced a skyrocketing growth of HCV infections because of paid blood donations in the late 1980s to early 1990s (Lu et al. 2013; Yin et al. 2015). Fortunately, the prevalence of HCV infection has declined dramatically since mid-1990s and was less than 1% in recent years (Cui and Jia 2013; Fu et al. 2010). However, the huge size of population in China makes a quite enormous absolute number of people infected with HCV. Like other agents causing blood-borne diseases, HCV is mainly transmitted through exchange of bodily fluids and intravenous drug use, as well as vertical transmission (Lauer and Walker 2001). Compared to the more in-depth understanding on infection and virus transmission in high-risk groups, limited data are available concerning the HCV infection caused by infrequent exposure in general population. Herein, we documented two sisters with chronic HCV infection and our attempts to dissect the transmission routes of their infections.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00183-4" class="mainColor" target="_blank">The Limitation of Rapid Tests for DENV2 Infection in Host with Unique Immune Status: Low NS1 Antigenemia and Deficient Antibody Responses</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Lei Yu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yingfen Wen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Mengrong Xiang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wenxin Hong</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lingzhai Zhao</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Fuchun Zhang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00183-4" target="_blank" class="list-doi">10.1007/s12250-019-00183-4</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>25 April 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00183-4" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00183-4" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/38bf1fed-c99f-4e5f-965e-f9b1ca70980e.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Dengue infections are caused by all four serotypes of dengue virus (DENV1&#8211;4). In recent years it has been confirmed that DENV-1 and DENV-2 are co-circulated in Guangzhou, Guangdong, China (Lai et al. 2015; Luo et al. 2017). DENV-2 strains were imported from Thailand and Indonesia (Luo et al. 2017; Zhao et al. 2014, 2016), and had become the second local circulated serotype in Guangzhou.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00187-0" class="mainColor" target="_blank">Visualizing the Transport of Porcine Reproductive and Respiratory Syndrome Virus in Live Cells by Quantum Dots-Based Single Virus Tracking</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Zhenpu Liang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Pengjuan Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Caiping Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Deepali Singh</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xiaoxia Zhang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00187-0" target="_blank" class="list-doi">10.1007/s12250-019-00187-0</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>20 August 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00187-0" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00187-0" target="_blank">Springerlink</a>        
      
                 
                       <a href="article/doi/10.1007/s12250-019-00187-0" target="_blank">ESM</a>
                    
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Quantum dots (QDs)-based single particle analysis technique enables real-time tracking of the viral infection in live cells with great sensitivity over a long period of time. The porcine reproductive and respiratory syndrome virus (PRRSV) is a small virus with the virion size of 40&#8211;60 nm which causes great economic losses to the swine industry worldwide. A clear understanding of the viral infection mechanism is essential for the development of effective antiviral strategies. In this study, we labeled the PRRSV with QDs using the streptavidin&#8211;biotin labeling system and monitored the viral infection process in live cells. Our results indicated that the labeling method had negligible effect on viral infectivity. We also observed that prior to the entry, PRRSV vibrated on the plasma membrane, and entered the cells via endosome mediated cell entry pathway. Viruses moved in a slow&#8211;fast&#8211;slow oscillatory movement pattern and finally accumulated in a perinuclear region of the cell. Our results also showed that once inside the cell, PRRSV moved along the microtubule, microfilament and vimentin cytoskeletal elements. During the transport process, virus particles also made contacts with non-muscle myosin heavy chain II-A (NMHC II-A), visualized as small spheres in cytoplasm. This study can facilitate the application of QDs in virus infection imaging, especially the smaller-sized viruses and provide some novel and important insights into PRRSV infection mechanism.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00181-6" class="mainColor" target="_blank">The Establishment of an In Vivo HIV-1 Infection Model in Humanized B-NSG Mice</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Tian-Jiao Fan</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Li Sun</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xian-Guang Yang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xia Jin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Wei-Wei Sun</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jian-Hua Wang</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00181-6" target="_blank" class="list-doi">10.1007/s12250-019-00181-6</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>12 July 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00181-6" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00181-6" target="_blank">Springerlink</a>        
      
                           
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Suitable animal models for human immunodeficiency virus type 1 (HIV-1) infection are important for elucidating viral pathogenesis and evaluating antiviral strategies in vivo. The B-NSG (NOD-PrkdcscidIl2rgtm1/Bcge) mice that have severe immune defect phenotype are examined for the suitability of such a model in this study. Human peripheral blood mononuclear cells (PBMCs) were engrafted into B-NSG mice via mouse tail vein injection, and the repopulated human T-lymphocytes were observed at as early as 3-weeks post-transplantation in mouse peripheral blood and several tissues. The humanized mice could be infected by HIV-1, and the infection recapitulated features of T-lymphocyte dynamic observed in HIV-1 infected humans, meanwhile the administration of combination antiretroviral therapy (cART) suppressed viral replication and restored T lymphocyte abnormalities. The establishment of HIV-1 infected humanized B-NSG mice not only provides a model to study virus and T cell interplays, but also can be a useful tool to evaluate antiviral strategies.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00173-6" class="mainColor" target="_blank">Pan-Genomic Analysis of African Swine Fever Virus</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Ziming Wang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lijia Jia</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing Li</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Haizhou Liu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Di Liu</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00173-6" target="_blank" class="list-doi">10.1007/s12250-019-00173-6</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>10 September 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00173-6" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00173-6" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/5a930eaf-4469-44c8-acaa-f101a62e6b6d.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">African swine fever (ASF) is a severe haemorrhagic fever in domestic pigs and wild boar with extremely high mortality rate. It is cataloged as a notifiable disease by the World Organization for Animal Health (OIE). The etiological agent that causes the highly lethal disease is the African swine fever virus (ASFV) (Sanchez-Vizcaino et al. 2015). ASFV is the only known member of the genus Asfivirus and family Asfarviridae. The family Asfarviridae belongs to the member of nucleocytoplasmic large DNA viruses (NCLDV) superfamily (Iyer et al. 2006; Costard et al. 2009). Overall, the ASFV virion presents an icosahedral morphology with a multilayered structure (Wang et al. 2019). The genome of ASFV is a large doublestranded DNA (dsDNA) molecule that varies in length from about 170 to 193 kilobase pairs and encodes between 150 and 167 open reading frames (ORFs) depending on the isolate (Dixon et al. 2013). In addition, ASFV also infects African wild suids, including warthogs (Phacochoerus africanus) and bushpigs (Potamochoerus larvatus), which act as asymptomatic carriers. Soft ticks of the Ornithodoros moubata complex also serve as a natural reservoir and transmit the disease to suids. In East Africa, ASFV is maintained in an ancient sylvatic cycle involving warthogs and soft ticks (Ornithodoros genus) that inhabit their burrows (Jori et al. 2013).</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00172-7" class="mainColor" target="_blank">Isolation and Characterization of A Novel Fowl Adenovirus Serotype 8a Strain from China</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Li Chen</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Lijuan Yin</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Peng Peng</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Qingfeng Zhou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yunping Du</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yun Zhang</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Chunyi Xue</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yongchang Cao</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00172-7" target="_blank" class="list-doi">10.1007/s12250-019-00172-7</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>04 June 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00172-7" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00172-7" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/b808dbc9-7a17-493d-943c-d703f4405ad1.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">Since 2012, the clinical cases of inclusion body hepatitis showed an increasing trend in China, causing considerable economic losses to the poultry industry. In this study, a fowl adenovirus strain CH/GDLZ/201801 was isolated from a chicken flock experiencing inclusion body hepatitis and analyzed by complete genome sequencing. The pathogenicity of the new virus strain was examined by experimental infection of specific pathogen free chickens. The isolate was identified by immunofluorescence and the virions presented typical icosahedral particles under transmission electron microscopy. The full genome of the isolate was 44,329 nucleotides in length with 58% G+C content. Phylogenetic analysis, based on the whole genome, revealed that the new isolate was closest to serotype 8a from the species Fowl aviadenovirus E (FAdVE). Recombination analysis and phylogenetic analysis showed that the new isolate is a recombinant strain between FAdV- 8a and FAdV-8b. In infection experiments, three infected chickens showed clinical signs and one chicken died on day 7 post infection, corresponding to 5% mortality. Macroscopic and microscopic lesions in the liver were observed, and viral antigen could be detected in the livers by immunohistochemical staining and TEM. Taken together, our study describes the genomic characteristics and pathogenicity of a FAdV-8a strain in China. It would lay a solid foundation for further study of the pathogenic mechanism and vaccine development of the virus.</span>
	        </p><p class="article-list-right clear">
			    		  
	    			
				<p class="article-list-title">
	           										           			<a href="article/doi/10.1007/s12250-019-00182-5" class="mainColor" target="_blank">Hepatitis C Virus NS2 Protein Suppresses RNA Interference in Cells</a>
		           											</p>
				<p class="article-list-author">
						                	                	<span>
	                    	<a href="">Hui Zhou</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Qi Qian</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Ting Shu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jiuyue Xu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jing Kong</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Jingfang Mu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Yang Qiu</a>, 	                    </span>
	                	                	<span>
	                    	<a href="">  Xi Zhou</a>	                    </span>
	                				</p>
				
			
		        <p class="article-list-time latest_info">
		        	 
		      	     		      	     doi: <a href="http://dx.doi.org/10.1007/s12250-019-00182-5" target="_blank" class="list-doi">10.1007/s12250-019-00182-5</a>
		      	     		      	</p>
		      	<p class="article-list-date">
            		            	
	            		Received: <span>12 October 2019</span>
	            	&#8195;
            	            	            	            </p>
		      	 <p class="article-list-zy clear">
		      	  
                                       
                   	   
			            			            <a href="/en/article/doi/10.1007/s12250-019-00182-5" target="_blank">Abstract</a>
			                          
                
        
            <a href="">PDF</a> 
        
                <a href="https://link.springer.com/article/10.1007/s12250-019-00182-5" target="_blank">Springerlink</a>        
      
                   <a href="/fileZGBDX/journal/article/file/87dea088-2317-4b2a-84f6-9f5eab3fab66.pdf" target="_blank">ESM</a>
                 
			 
			 
			 
				      	
		    </p>
		    
		    </p><p class="article-list-zy morenstyle">
				 
				<span class="search-article-abstract">RNAi interference (RNAi) is an evolutionarily conserved post-transcriptional gene silencing mechanism and has been well recognized as an important antiviral immunity in eukaryotes. Numerous viruses have been shown to encode viral suppressors of RNAi (VSRs) to antagonize antiviral RNAi. Hepatitis C virus (HCV) is a medically important human pathogen that causes acute and chronic hepatitis. In this study, we screened all the nonstructural proteins of HCV and found that HCV NS2 could suppress RNAi induced either by small hairpin RNAs (shRNAs) or small interfering RNAs (siRNAs) in mammalian cells. Moreover, we demonstrated that NS2 could suppress RNAi via its direct interaction with double-stranded RNAs (dsRNAs) and siRNAs, and further identified that the cysteine 184 of NS2 is required for the RNAi suppression activity through a serial of point mutation analyses. Together, our findings uncovered that HCV NS2 can act as a VSR in vitro, thereby providing novel insights into the life cycle and virus-host interactions of HCV.</span>
	        </p><ul class="column-ul">
                                                                                                                        <li class="mainColor"><a href="#c_1" class="mainColor">Perspectives</a></li>
                                                                                                                                                                <li class="mainColor"><a href="#c_2" class="mainColor">Perspective</a></li>
                                                                                                                                                                <li class="mainColor"><a href="#c_3" class="mainColor">Review</a></li>
                                                                                                                                                                <li class="mainColor"><a href="#c_4" class="mainColor">Research Article</a></li>
                                                                                                                                                                <li class="mainColor"><a href="#c_5" class="mainColor">Letter</a></li>
                                                                                                                    </ul><p class="toolBox2 clearfix fr">
                    
                        Select All
                    
                    <p class="derived fl">
                        |&#160;&#160;
                        &#160;&#160;
                        
                    </p>
                </p><p class="article-list-journalg" id="c_1">
    <a href="">Perspectives</a>
</p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00204-7" class="mainColor" target="_blank">Old Weapon for New Enemy: Drug Repurposing for Treatment of Newly Emerging Viral Diseases</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Deyin Guo</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 253-255.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00204-7" target="_blank" class="list-doi">10.1007/s12250-020-00204-7</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>29 January 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>31 January 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>11 February 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00204-7" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 196KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00204-7" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                            
        
                 
        <p class="citation">
            Deyin Guo. Old Weapon for New Enemy: Drug Repurposing for Treatment of Newly Emerging Viral Diseases[J]. Virologica Sinica, 2020, 35(3): 253-255. doi: 10.1007/s12250-020-00204-7.
        </p>
    </p><p class="article-list-journalg" id="c_2">
    <a href="">Perspective</a>
</p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00205-6" class="mainColor" target="_blank">Compensation of ACE2 Function for Possible Clinical Management of 2019-nCoV-Induced Acute Lung Injury</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Yuntao Wu</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 256-258.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00205-6" target="_blank" class="list-doi">10.1007/s12250-020-00205-6</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>01 February 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>03 February 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>07 February 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00205-6" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 261KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00205-6" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Yuntao Wu. Compensation of ACE2 Function for Possible Clinical Management of 2019-nCoV-Induced Acute Lung Injury[J]. Virologica Sinica, 2020, 35(3): 256-258. doi: 10.1007/s12250-020-00205-6.
        </p>
    </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00243-0" class="mainColor" target="_blank">mRNA Vaccines: Possible Tools to Combat SARS-CoV-2</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Changhua Yi</a>, 				</span>
                                <span>
                	<a href=""> Yongxiang Yi</a>, 				</span>
                                <span>
                	<a href=""> Junwei Li</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 259-262.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00243-0" target="_blank" class="list-doi">10.1007/s12250-020-00243-0</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>20 March 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>18 May 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>10 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00243-0" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 258KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00243-0" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Changhua Yi, Yongxiang Yi and Junwei Li. mRNA Vaccines: Possible Tools to Combat SARS-CoV-2[J]. Virologica Sinica, 2020, 35(3): 259-262. doi: 10.1007/s12250-020-00243-0.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">Since December 2019, a  novel coronavirus pneumonia outbreak has been reported in 212 countries and territories, with over 5, 000, 000 infections and over 300, 000 deaths on May 22, 2020. To date, except some repurposed drugs have been shown inhibitory effect on the so-called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), no drugs have been licensed to be effective and safe in treating COVID-19. Inspite of emergent need, no authorized vaccines have been approved for clinical use against this new virus currently. After the genetic map of SARS-CoV-2 was shared, scientists and biotech and vaccine companies promptly launched a race to pursue different types of vaccines to prevent illness from this novel virus. Among these vaccine projects, mRNA vaccines are showing a promising future with their feature of fast development.</span>
            </p><p class="article-list-journalg" id="c_3">
    <a href="">Review</a>
</p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00206-5" class="mainColor" target="_blank">The First Disease X is Caused by a Highly Transmissible Acute Respiratory Syndrome Coronavirus</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Shibo Jiang</a>, 				</span>
                                <span>
                	<a href=""> Zheng-Li Shi</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 263-265.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00206-5" target="_blank" class="list-doi">10.1007/s12250-020-00206-5</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>06 February 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>09 February 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>14 February 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00206-5" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 207KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00206-5" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                            
        
                 
        <p class="citation">
            Shibo Jiang and Zheng-Li Shi. The First Disease X is Caused by a Highly Transmissible Acute Respiratory Syndrome Coronavirus[J]. Virologica Sinica, 2020, 35(3): 263-265. doi: 10.1007/s12250-020-00206-5.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">Based on the announcement of the World Health Organization (WHO) in 2018,  the Wuhan pneumonia caused by an unknown etiology should be recognized as the first Disease X. Later, the pathogen was identified to be a novel coronavirus denoted 2019-nCoV, which has 79.5% and 96% whole genome sequence identify to SARS-CoV and bat SARS-related coronavirus (SARSr-CoV-RaTG13), respectively, suggesting its potential bat origin. With high human-to-human transmission rate (R0), 2019-nCoV has quickly spread in China and other countries, resulting in 34, 953 confirmed cases and 725 deaths as of 8 February 2020,  thus calling for urgent development of therapeutics and prophylactics. Here we suggest renaming 2019-nCoV as "transmissible acute respiratory syndrome coronavirus (TARS-CoV)" and briefly review the advancement of research and development of neutralizing antibodies and vaccines targeting the receptor-binding domain (RBD) and viral fusion inhibitors targeting the heptad repeat 1 (HR1) domain in spike protein of 2019-nCoV.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00207-4" class="mainColor" target="_blank">Understanding SARS-CoV-2-Mediated Inflammatory Responses: From Mechanisms to Potential Therapeutic Tools</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Yajing Fu</a>, 				</span>
                                <span>
                	<a href=""> Yuanxiong Cheng</a>, 				</span>
                                <span>
                	<a href=""> Yuntao Wu</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 266-271.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00207-4" target="_blank" class="list-doi">10.1007/s12250-020-00207-4</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>14 February 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>16 February 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>03 March 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00207-4" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 420KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00207-4" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Yajing Fu, Yuanxiong Cheng and Yuntao Wu. Understanding SARS-CoV-2-Mediated Inflammatory Responses: From Mechanisms to Potential Therapeutic Tools[J]. Virologica Sinica, 2020, 35(3): 266-271. doi: 10.1007/s12250-020-00207-4.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">Currently there is no effective antiviral therapy for SARS-CoV-2 infection, which frequently leads to fatal inflammatory responses and acute lung injury. Here, we discuss the various mechanisms of SARS-CoV-mediated inflammation. We also assume that SARS-CoV-2 likely shares similar inflammatory responses. Potential therapeutic tools to reduce SARS-CoV-2-induced inflammatory responses include various methods to block FcR activation. In the absence of a proven clinical FcR blocker, the use of intravenous immunoglobulin to block FcR activation may be a viable option for the urgent treatment of pulmonary inflammation to prevent severe lung injury. Such treatment may also be combined with systemic anti-inflammatory drugs or corticosteroids. However, these strategies, as proposed here, remain to be clinically tested for effectiveness.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00219-0" class="mainColor" target="_blank">The Rapid Assessment and Early Warning Models for COVID-19</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Zhihua Bai</a>, 				</span>
                                <span>
                	<a href=""> Yue Gong</a>, 				</span>
                                <span>
                	<a href=""> Xiaodong Tian</a>, 				</span>
                                <span>
                	<a href=""> Ying Cao</a>, 				</span>
                                <span>
                	<a href=""> Wenjun Liu</a>, 				</span>
                                <span>
                	<a href=""> Jing Li</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 272-279.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00219-0" target="_blank" class="list-doi">10.1007/s12250-020-00219-0</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>08 March 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>27 March 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>01 April 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00219-0" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 482KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00219-0" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Zhihua Bai, Yue Gong, Xiaodong Tian, Ying Cao, Wenjun Liu and Jing Li. The Rapid Assessment and Early Warning Models for COVID-19[J]. Virologica Sinica, 2020, 35(3): 272-279. doi: 10.1007/s12250-020-00219-0.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">Human beings have experienced a serious public health event as the new pneumonia (COVID-19), caused by the severe acute respiratory syndrome coronavirus has killed more than 3000 people in China, most of them elderly or people with underlying chronic diseases or immunosuppressed states. Rapid assessment and early warning are essential for outbreak analysis in response to serious public health events. This paper reviews the current model analysis methods and conclusions from both micro and macro perspectives. The establishment of a comprehensive assessment model, and the use of model analysis prediction, is very efficient for the early warning of infectious diseases. This would significantly improve global surveillance capacity, particularly in developing regions, and improve basic training in infectious diseases and molecular epidemiology.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00244-z" class="mainColor" target="_blank">Conditionally Reprogrammed Human Normal Airway Epithelial Cells at ALI: A Physiological Model for Emerging Viruses</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Xuefeng Liu</a>, 				</span>
                                <span>
                	<a href=""> Yuntao Wu</a>, 				</span>
                                <span>
                	<a href=""> Lijun Rong</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 280-289.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00244-z" target="_blank" class="list-doi">10.1007/s12250-020-00244-z</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>16 April 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>22 May 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>17 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00244-z" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 680KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00244-z" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Xuefeng Liu, Yuntao Wu and Lijun Rong. Conditionally Reprogrammed Human Normal Airway Epithelial Cells at ALI: A Physiological Model for Emerging Viruses[J]. Virologica Sinica, 2020, 35(3): 280-289. doi: 10.1007/s12250-020-00244-z.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">Cancer cell lines have been used widely in cancer biology, and as biological or functional cell systems in many biomedical research fields. These cells are usually defective for many normal activities or functions due to significant genetic and epigenetic changes. Normal primary cell yields and viability from any original tissue specimens are usually relatively low or highly variable. These normal cells cease after a few passages or population doublings due to very limited proliferative capacity. Animal models (ferret, mouse, etc.) are often used to study virus-host interaction. However, viruses usually need to be adapted to the animals by several passages due to tropism restrictions including viral receptors and intracellular restrictions. Here we summarize applications of conditionally reprogrammed cells (CRCs), long-term cultures of normal airway epithelial cells from human nose to lung generated by conditional cell reprogramming (CR) technology, as an ex vivo model in studies of emerging viruses. CR allows to robustly propagate cells from non-invasive or minimally invasive specimens, for example, nasal or endobronchial brushing. This process is rapid (2 days) and conditional. The CRCs maintain their differentiation potential and lineage functions, and have been used for studies of adenovirus, rhinovirus, respiratory syncytial virus, influenza viruses, parvovirus, and SARS-CoV. The CRCs can be easily used for air-liquid interface (ALI) polarized 3D cultures, and these coupled CRC/ALI cultures mimic physiological conditions and are suitable for studies of viral entry including receptor binding and internalization, innate immune responses, viral replications, and drug discovery as an ex vivo model for emerging viruses.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00252-z" class="mainColor" target="_blank">A Comprehensive Review of Animal Models for Coronaviruses: SARSCoV-2, SARS-CoV, and MERS-CoV</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Ashutosh Singh</a>, 				</span>
                                <span>
                	<a href=""> Rahul Soloman Singh</a>, 				</span>
                                <span>
                	<a href=""> Phulen Sarma</a>, 				</span>
                                <span>
                	<a href=""> Gitika Batra</a>, 				</span>
                                <span>
                	<a href=""> Rupa Joshi</a>, 				</span>
                                <span>
                	<a href=""> Hardeep Kaur</a>, 				</span>
                                <span>
                	<a href=""> Amit Raj Sharma</a>, 				</span>
                                <span>
                	<a href=""> Ajay Prakash</a>, 				</span>
                                <span>
                	<a href=""> Bikash Medhi</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 290-304.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00252-z" target="_blank" class="list-doi">10.1007/s12250-020-00252-z</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>25 April 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>02 June 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>30 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00252-z" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 337KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00252-z" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                            
        
                 
        <p class="citation">
            Ashutosh Singh, Rahul Soloman Singh, Phulen Sarma, Gitika Batra, Rupa Joshi, Hardeep Kaur, Amit Raj Sharma, Ajay Prakash and Bikash Medhi. A Comprehensive Review of Animal Models for Coronaviruses: SARSCoV-2, SARS-CoV, and MERS-CoV[J]. Virologica Sinica, 2020, 35(3): 290-304. doi: 10.1007/s12250-020-00252-z.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">The recent outbreak of coronavirus disease (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has already affected a large population of the world. SARS-CoV-2 belongs to the same family of severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). COVID-19 has a complex pathology involving severe acute respiratory infection, hyper-immune response, and coagulopathy. At present, there is no therapeutic drug or vaccine approved for the disease. There is an urgent need for an ideal animal model that can reflect clinical symptoms and underlying etiopathogenesis similar to COVID-19 patients which can be further used for evaluation of underlying mechanisms, potential vaccines, and therapeutic strategies. The current review provides a paramount insight into the available animal models of SARS-CoV-2, SARS-CoV, and MERS-CoV for the management of the diseases.</span>
            </p><p class="article-list-journalg" id="c_4">
    <a href="">Research Article</a>
</p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00227-0" class="mainColor" target="_blank">Clinical Manifestation and Laboratory Characteristics of SARS-CoV-2 Infection in Pregnant Women</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Chunchen Wu</a>, 				</span>
                                <span>
                	<a href=""> Wenzhong Yang</a>, 				</span>
                                <span>
                	<a href=""> Xiaoxue Wu</a>, 				</span>
                                <span>
                	<a href=""> Tianzhu Zhang</a>, 				</span>
                                <span>
                	<a href=""> Yaoyao Zhao</a>, 				</span>
                                <span>
                	<a href=""> Wei Ren</a>, 				</span>
                                <span>
                	<a href=""> Jianbo Xia</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 305-310.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00227-0" target="_blank" class="list-doi">10.1007/s12250-020-00227-0</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>06 March 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>10 April 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>20 April 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00227-0" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 1185KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00227-0" target="_blank">Springerlink</a>

                	                
	                    	                    <a href="article/doi/10.1007/s12250-020-00227-0#Supplements" target="_blank">ESM</a>
	                    	                
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Chunchen Wu, Wenzhong Yang, Xiaoxue Wu, Tianzhu Zhang, Yaoyao Zhao, Wei Ren and Jianbo Xia. Clinical Manifestation and Laboratory Characteristics of SARS-CoV-2 Infection in Pregnant Women[J]. Virologica Sinica, 2020, 35(3): 305-310. doi: 10.1007/s12250-020-00227-0.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) epidemic has become a major challenge to public health in China and other countries, considering its pathogenicity across all age groups. Pregnancy is a unique physiological condition, and is characterized by altered immunity and elevated hormone levels to actively tolerate the semi-allogeneic fetus, which undergoes a sudden and substantial fluctuation during the immediate postpartum period. Changes in clinical features, laboratory characteristics, and imaging features of pregnant women during the pre-partum and post-partum periods require further elucidation. Here, we retrospectively analyzed the clinical features, laboratory characteristics, and imaging features of eight pregnant cases of SARS-CoV-2 infection during the pre-partum and post-partum periods. Our results showed that four of the eight pregnant women were asymptomatic before delivery but became symptomatic post-partum. Correspondingly, white blood cell (WBC) counts increased and lymphocyte (LYMPH) counts decreased. C-reactive protein (CRP) levels in the serum also increased to a higher level than those in general pregnancy. Therefore, it is imperative to closely monitor laboratory parameters including the WBC count, LYMPH count, and CRP, along with other imaging features in chest CT scans, to promptly prevent, diagnose, and treat a SARS-CoV-2 infection during pregnancy.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00253-y" class="mainColor" target="_blank">Long Term Culture of Human Kidney Proximal Tubule Epithelial Cells Maintains Lineage Functions and Serves as an Ex vivo Model for Coronavirus Associated Kidney Injury</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Siyu Xia</a>, 				</span>
                                <span>
                	<a href=""> Ming Wu</a>, 				</span>
                                <span>
                	<a href=""> Si Chen</a>, 				</span>
                                <span>
                	<a href=""> Tao Zhang</a>, 				</span>
                                <span>
                	<a href=""> Lina Ye</a>, 				</span>
                                <span>
                	<a href=""> Jun Liu</a>, 				</span>
                                <span>
                	<a href=""> Hui Li</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 311-320.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00253-y" target="_blank" class="list-doi">10.1007/s12250-020-00253-y</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>26 March 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>09 June 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>29 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00253-y" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 2087KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00253-y" target="_blank">Springerlink</a>

                                	<a href="/fileZGBDX/journal/article/file/fac3f7a8-8d54-4845-98ae-d13a0f6fa63e.pdf" target="_blank">ESM</a>
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Siyu Xia, Ming Wu, Si Chen, Tao Zhang, Lina Ye, Jun Liu and Hui Li. Long Term Culture of Human Kidney Proximal Tubule Epithelial Cells Maintains Lineage Functions and Serves as an Ex vivo Model for Coronavirus Associated Kidney Injury[J]. Virologica Sinica, 2020, 35(3): 311-320. doi: 10.1007/s12250-020-00253-y.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">The mechanism of how SARS-CoV-2 causes severe multi-organ failure is largely unknown. Acute kidney injury (AKI) is one of the frequent organ damage in severe COVID-19 patients. Previous studies have shown that human renal tubule cells could be the potential host cells targeted by SARS-CoV-2. Traditional cancer cell lines or immortalized cell lines are genetically and phenotypically different from host cells. Animal models are widely used, but often fail to reflect a physiological and pathogenic status because of species tropisms. There is an unmet need for normal human epithelial cells for disease modeling. In this study, we successfully established long term cultures of normal human kidney proximal tubule epithelial cells (KPTECs) in 2D and 3D culture systems using conditional reprogramming (CR) and organoids techniques. These cells had the ability to differentiate and repair DNA damage, and showed no transforming property. Importantly, the CR KPTECs maintained lineage function with expression of specific transporters (SLC34A3 and cubilin). They also expressed angiotensin-converting enzyme 2 (ACE2), a receptor for SARS-CoV and SARS-CoV-2. In contrast, cancer cell line did not express endogenous SLC34A3, cubilin and ACE2. Very interestingly, ACE2 expression was around twofold higher in 3D organoids culture compared to that in 2D CR culture condition. Pseudovirion assays demonstrated that SARS-CoV spike (S) protein was able to enter CR cells with luciferase reporter. This integrated 2D CR and 3D organoid cultures provide a physiological ex vivo model to study kidney functions, innate immune response of kidney cells to viruses, and a novel platform for drug discovery and safety evaluation.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00242-1" class="mainColor" target="_blank">SARS-Coronavirus-2 Nsp13 Possesses NTPase and RNA Helicase Activities That Can Be Inhibited by Bismuth Salts</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Ting Shu</a>, 				</span>
                                <span>
                	<a href=""> Muhan Huang</a>, 				</span>
                                <span>
                	<a href=""> Di Wu</a>, 				</span>
                                <span>
                	<a href=""> Yujie Ren</a>, 				</span>
                                <span>
                	<a href=""> Xueyi Zhang</a>, 				</span>
                                <span>
                	<a href=""> Yang Han</a>, 				</span>
                                <span>
                	<a href=""> Jingfang Mu</a>, 				</span>
                                <span>
                	<a href=""> Ruibing Wang</a>, 				</span>
                                <span>
                	<a href=""> Yang Qiu</a>, 				</span>
                                <span>
                	<a href=""> Ding-Yu Zhang</a>, 				</span>
                                <span>
                	<a href=""> Xi Zhou</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 321-329.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00242-1" target="_blank" class="list-doi">10.1007/s12250-020-00242-1</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>07 May 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>19 May 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>04 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00242-1" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 781KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00242-1" target="_blank">Springerlink</a>

                	                
	                    	                    <a href="article/doi/10.1007/s12250-020-00242-1#Supplements" target="_blank">ESM</a>
	                    	                
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Ting Shu, Muhan Huang, Di Wu, Yujie Ren, Xueyi Zhang, Yang Han, Jingfang Mu, Ruibing Wang, Yang Qiu, Ding-Yu Zhang and Xi Zhou. SARS-Coronavirus-2 Nsp13 Possesses NTPase and RNA Helicase Activities That Can Be Inhibited by Bismuth Salts[J]. Virologica Sinica, 2020, 35(3): 321-329. doi: 10.1007/s12250-020-00242-1.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">The ongoing outbreak of Coronavirus Disease 2019 (COVID-19) has become a global public health emergency. SARS-coronavirus-2 (SARS-CoV-2), the causative pathogen of COVID-19, is a positive-sense single-stranded RNA virus belonging to the family Coronaviridae. For RNA viruses, virus-encoded RNA helicases have long been recognized to play pivotal roles during viral life cycles by facilitating the correct folding and replication of viral RNAs. Here, our studies show that SARS-CoV-2-encoded nonstructural protein 13 (nsp13) possesses the nucleoside triphosphate hydrolase (NTPase) and RNA helicase activities that can hydrolyze all types of NTPs and unwind RNA helices dependently of the presence of NTP, and further characterize the biochemical characteristics of these two enzymatic activities associated with SARS-CoV-2 nsp13. Moreover, we found that some bismuth salts could effectively inhibit both the NTPase and RNA helicase activities of SARS-CoV-2 nsp13 in a dose-dependent manner. Thus, our findings demonstrate the NTPase and helicase activities of SARS-CoV-2 nsp13, which may play an important role in SARS-CoV-2 replication and serve as a target for antivirals.</span>
            </p><p class="article-list-journalg" id="c_5">
    <a href="">Letter</a>
</p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00203-8" class="mainColor" target="_blank">Clinical Features and Treatment of 2019-nCov Pneumonia Patients in Wuhan: Report of A Couple Cases</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Zhan Zhang</a>, 				</span>
                                <span>
                	<a href=""> Xiaochen Li</a>, 				</span>
                                <span>
                	<a href=""> Wei Zhang</a>, 				</span>
                                <span>
                	<a href=""> Zheng-Li Shi</a>, 				</span>
                                <span>
                	<a href=""> Zhishui Zheng</a>, 				</span>
                                <span>
                	<a href=""> Tao Wang</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 330-336.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00203-8" target="_blank" class="list-doi">10.1007/s12250-020-00203-8</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>25 January 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>28 January 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>07 February 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00203-8" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 3888KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00203-8" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Zhan Zhang, Xiaochen Li, Wei Zhang, Zheng-Li Shi, Zhishui Zheng and Tao Wang. Clinical Features and Treatment of 2019-nCov Pneumonia Patients in Wuhan: Report of A Couple Cases[J]. Virologica Sinica, 2020, 35(3): 330-336. doi: 10.1007/s12250-020-00203-8.
        </p>
    </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00212-7" class="mainColor" target="_blank">A Unique Protease Cleavage Site Predicted in the Spike Protein of the Novel Pneumonia Coronavirus (2019-nCoV) Potentially Related to Viral Transmissibility</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Qiong Wang</a>, 				</span>
                                <span>
                	<a href=""> Ye Qiu</a>, 				</span>
                                <span>
                	<a href=""> Jin-Yan Li</a>, 				</span>
                                <span>
                	<a href=""> Zhi-Jian Zhou</a>, 				</span>
                                <span>
                	<a href=""> Ce-Heng Liao</a>, 				</span>
                                <span>
                	<a href=""> Xing-Yi Ge</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 337-339.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00212-7" target="_blank" class="list-doi">10.1007/s12250-020-00212-7</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>15 February 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>06 March 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>20 March 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00212-7" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 562KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00212-7" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Qiong Wang, Ye Qiu, Jin-Yan Li, Zhi-Jian Zhou, Ce-Heng Liao and Xing-Yi Ge. A Unique Protease Cleavage Site Predicted in the Spike Protein of the Novel Pneumonia Coronavirus (2019-nCoV) Potentially Related to Viral Transmissibility[J]. Virologica Sinica, 2020, 35(3): 337-339. doi: 10.1007/s12250-020-00212-7.
        </p>
    </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00214-5" class="mainColor" target="_blank">Inefficiency of Sera from Mice Treated with Pseudotyped SARS-CoV to Neutralize 2019-nCoV Infection</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Zezhong Liu</a>, 				</span>
                                <span>
                	<a href=""> Shuai Xia</a>, 				</span>
                                <span>
                	<a href=""> Xinling Wang</a>, 				</span>
                                <span>
                	<a href=""> Qiaoshuai Lan</a>, 				</span>
                                <span>
                	<a href=""> Wei Xu</a>, 				</span>
                                <span>
                	<a href=""> Qian Wang</a>, 				</span>
                                <span>
                	<a href=""> Shibo Jiang</a>, 				</span>
                                <span>
                	<a href=""> Lu Lu</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 340-343.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00214-5" target="_blank" class="list-doi">10.1007/s12250-020-00214-5</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>22 February 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>14 March 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>31 March 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00214-5" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 1319KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00214-5" target="_blank">Springerlink</a>

                                	<a href="/fileZGBDX/journal/article/file/9e85b38f-70c2-4e24-8053-dfb4d16f66ba.pdf" target="_blank">ESM</a>
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Zezhong Liu, Shuai Xia, Xinling Wang, Qiaoshuai Lan, Wei Xu, Qian Wang, Shibo Jiang and Lu Lu. Inefficiency of Sera from Mice Treated with Pseudotyped SARS-CoV to Neutralize 2019-nCoV Infection[J]. Virologica Sinica, 2020, 35(3): 340-343. doi: 10.1007/s12250-020-00214-5.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">The novel coronavirus 2019-nCoV has caused the pandemic of Wuhan pneumonia recently, posing a serious threat to global public health, and thus calling for the development of therapeutics and prophylactics. Here we showed that high titer anti-SARS-CoV spike protein serum cannot effectively neutralize 2019-nCoV infection. Based on our previous research, we developed SARS pseudovirus (SARS-PsV) and MERS pseudovirus (MERS-PsV) as immunogens to immunize mice. We found sera from mice treated with SARS-CoV S protein could potently cross-neutralize infection by SARS-CoV (50% neutralizing antibody titers, NT50 &gt; 40, 000) and SARS-related coronavirus (NT50 &gt; 7, 000), but weakly for 2019-nCoV infection NT50 &lt; 100), implying that it may not be practical to treat 2019-nCoV infection with anti-SARS-CoV antibodies and that people with history of SARS-CoV infection many years ago may not be resistant to 2019-nCoV infection.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00218-1" class="mainColor" target="_blank">Development of a Novel Reverse Transcription Loop-Mediated Isothermal Amplification Method for Rapid Detection of SARS-CoV-2</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Renfei Lu</a>, 				</span>
                                <span>
                	<a href=""> Xiuming Wu</a>, 				</span>
                                <span>
                	<a href=""> Zhenzhou Wan</a>, 				</span>
                                <span>
                	<a href=""> Yingxue Li</a>, 				</span>
                                <span>
                	<a href=""> Lulu Zuo</a>, 				</span>
                                <span>
                	<a href=""> Jianru Qin</a>, 				</span>
                                <span>
                	<a href=""> Xia Jin</a>, 				</span>
                                <span>
                	<a href=""> Chiyu Zhang</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 344-347.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00218-1" target="_blank" class="list-doi">10.1007/s12250-020-00218-1</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>19 February 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>22 March 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>01 April 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00218-1" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 465KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00218-1" target="_blank">Springerlink</a>

                	                
	                    	                    <a href="article/doi/10.1007/s12250-020-00218-1#Supplements" target="_blank">ESM</a>
	                    	                
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Renfei Lu, Xiuming Wu, Zhenzhou Wan, Yingxue Li, Lulu Zuo, Jianru Qin, Xia Jin and Chiyu Zhang. Development of a Novel Reverse Transcription Loop-Mediated Isothermal Amplification Method for Rapid Detection of SARS-CoV-2[J]. Virologica Sinica, 2020, 35(3): 344-347. doi: 10.1007/s12250-020-00218-1.
        </p>
    </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00241-2" class="mainColor" target="_blank">Isolation and Growth Characteristics of SARS-CoV-2 in Vero Cell</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Pingping Yao</a>, 				</span>
                                <span>
                	<a href=""> Yachun Zhang</a>, 				</span>
                                <span>
                	<a href=""> Yisheng Sun</a>, 				</span>
                                <span>
                	<a href=""> Yulin Gu</a>, 				</span>
                                <span>
                	<a href=""> Fang Xu</a>, 				</span>
                                <span>
                	<a href=""> Bo Su</a>, 				</span>
                                <span>
                	<a href=""> Chen Chen</a>, 				</span>
                                <span>
                	<a href=""> Hangjing Lu</a>, 				</span>
                                <span>
                	<a href=""> Dehui Wang</a>, 				</span>
                                <span>
                	<a href=""> Zhangnv Yang</a>, 				</span>
                                <span>
                	<a href=""> Biao Niu</a>, 				</span>
                                <span>
                	<a href=""> Jiancai Chen</a>, 				</span>
                                <span>
                	<a href=""> Lixia Xie</a>, 				</span>
                                <span>
                	<a href=""> Lei Chen</a>, 				</span>
                                <span>
                	<a href=""> Yajing Zhang</a>, 				</span>
                                <span>
                	<a href=""> Hui Wang</a>, 				</span>
                                <span>
                	<a href=""> Yuying Zhao</a>, 				</span>
                                <span>
                	<a href=""> Yue Guo</a>, 				</span>
                                <span>
                	<a href=""> Juncheng Ruan</a>, 				</span>
                                <span>
                	<a href=""> Zhiyong Zhu</a>, 				</span>
                                <span>
                	<a href=""> Zhenfang Fu</a>, 				</span>
                                <span>
                	<a href=""> Dayong Tian</a>, 				</span>
                                <span>
                	<a href=""> Qi An</a>, 				</span>
                                <span>
                	<a href=""> Jianmin Jiang</a>, 				</span>
                                <span>
                	<a href=""> Hanping Zhu</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 348-350.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00241-2" target="_blank" class="list-doi">10.1007/s12250-020-00241-2</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>01 April 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>19 May 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>19 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00241-2" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 780KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00241-2" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Pingping Yao, Yachun Zhang, Yisheng Sun, Yulin Gu, Fang Xu, Bo Su, Chen Chen, Hangjing Lu, Dehui Wang, Zhangnv Yang, Biao Niu, Jiancai Chen, Lixia Xie, Lei Chen, Yajing Zhang, Hui Wang, Yuying Zhao, Yue Guo, Juncheng Ruan, Zhiyong Zhu, Zhenfang Fu, Dayong Tian, Qi An, Jianmin Jiang and Hanping Zhu. Isolation and Growth Characteristics of SARS-CoV-2 in Vero Cell[J]. Virologica Sinica, 2020, 35(3): 348-350. doi: 10.1007/s12250-020-00241-2.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">The coronavirus disease 2019 (COVID-19) broke out in early December 2019 in Wuhan, China and escalated into a global pandemic. There is an urgent need to understand the biology of SARS-CoV-2. In this letter, we report the isolation and characterization of seven isolates of SARS-CoV-2. Results show that our viruses have 99% sequence identity with published virus sequences. In addition, all viruses grew well in Vero cells, and one of the viruses had a deletion mutation after short passage. These results shall facilitate the understanding of the characteristics of SARS-CoV-2 in vitro.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00236-z" class="mainColor" target="_blank">A DNA Aptamer Based Method for Detection of SARS-CoV-2 Nucleocapsid Protein</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Zhiqiang Chen</a>, 				</span>
                                <span>
                	<a href=""> Qihan Wu</a>, 				</span>
                                <span>
                	<a href=""> Jing Chen</a>, 				</span>
                                <span>
                	<a href=""> Xiaohua Ni</a>, 				</span>
                                <span>
                	<a href=""> Jianfeng Dai</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 351-354.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00236-z" target="_blank" class="list-doi">10.1007/s12250-020-00236-z</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>16 April 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>30 April 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>25 May 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00236-z" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 530KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00236-z" target="_blank">Springerlink</a>

                	                
	                    	                    <a href="article/doi/10.1007/s12250-020-00236-z#Supplements" target="_blank">ESM</a>
	                    	                
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Zhiqiang Chen, Qihan Wu, Jing Chen, Xiaohua Ni and Jianfeng Dai. A DNA Aptamer Based Method for Detection of SARS-CoV-2 Nucleocapsid Protein[J]. Virologica Sinica, 2020, 35(3): 351-354. doi: 10.1007/s12250-020-00236-z.
        </p>
    </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00251-0" class="mainColor" target="_blank">SARS-CoV-2 Does Not Replicate in Aedes Mosquito Cells nor Present in Field-Caught Mosquitoes from Wuhan</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Han Xia</a>, 				</span>
                                <span>
                	<a href=""> Evans Atoni</a>, 				</span>
                                <span>
                	<a href=""> Lu Zhao</a>, 				</span>
                                <span>
                	<a href=""> Nanjie Ren</a>, 				</span>
                                <span>
                	<a href=""> Doudou Huang</a>, 				</span>
                                <span>
                	<a href=""> Rongjuan Pei</a>, 				</span>
                                <span>
                	<a href=""> Zhen Chen</a>, 				</span>
                                <span>
                	<a href=""> Jin Xiong</a>, 				</span>
                                <span>
                	<a href=""> Raphael Nyaruaba</a>, 				</span>
                                <span>
                	<a href=""> Shuqi Xiao</a>, 				</span>
                                <span>
                	<a href=""> Bo Zhang</a>, 				</span>
                                <span>
                	<a href=""> Zhiming Yuan</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 355-358.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00251-0" target="_blank" class="list-doi">10.1007/s12250-020-00251-0</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>10 May 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>08 June 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>29 June 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00251-0" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 741KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00251-0" target="_blank">Springerlink</a>

                                	<a href="/fileZGBDX/journal/article/file/da76d07c-b648-46b1-ba89-17179a4258dd.pdf" target="_blank">ESM</a>
                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                                 
                 
                            
        
                 
        <p class="citation">
            Han Xia, Evans Atoni, Lu Zhao, Nanjie Ren, Doudou Huang, Rongjuan Pei, Zhen Chen, Jin Xiong, Raphael Nyaruaba, Shuqi Xiao, Bo Zhang and Zhiming Yuan. SARS-CoV-2 Does Not Replicate in Aedes Mosquito Cells nor Present in Field-Caught Mosquitoes from Wuhan[J]. Virologica Sinica, 2020, 35(3): 355-358. doi: 10.1007/s12250-020-00251-0.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">With the rapid spread of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and growing fear, people have become very concerned about whether this novel coronavirus could be transmitted by mosquitoes. We evaluate the infectivity of SARS-CoV-2 in Aedes albopictus and Aedes aegypti derived cell lines. The results indicated that SARS-CoV-2 could not replicate in both C6/36, Sf9 and Aag2 cells. Further, no SARS-CoV-2 RNA was detected in the field collected Culex, and Anopheles mosquitoes during the months of April and May in Wuhan in 2020. Our findings highlight the restricted replication of SARS-CoV-2 in mosquito cells and field-caught mosquitoes.</span>
            </p><p class="article-list-right clear">
                         
                        
            <p class="article-list-title">
                                                <a href="article/doi/10.1007/s12250-020-00221-6" class="mainColor" target="_blank">Computational Identification of Small Interfering RNA Targets in SARS-CoV-2</a>
                                            </p>
            <p class="article-list-author">
                                                <span>
                	<a href="">Wei Chen</a>, 				</span>
                                <span>
                	<a href=""> Pengmian Feng</a>, 				</span>
                                <span>
                	<a href=""> Kewei Liu</a>, 				</span>
                                <span>
                	<a href=""> Meng Wu</a>, 				</span>
                                <span>
                	<a href=""> Hao Lin</a>				</span>
                            </p>
            <p class="article-list-time">
                2020, 35(3): 359-361.&#160;
                                
                    doi: <a href="http://dx.doi.org/10.1007/s12250-020-00221-6" target="_blank" class="list-doi">10.1007/s12250-020-00221-6</a>
                
                            </p>
            <p class="article-list-date">
            		            	
	            		Received: <span>06 March 2020</span>
	            	&#8195;
            	            		            	
	            		Accepted: <span>03 April 2020</span>
	            	&#8195;
            	            		            	
	            		Published: <span>15 April 2020</span>
	            	
            	            </p>
            <p class="article-list-zy clear">
                 
                                
                                                            <a href="article/doi/10.1007/s12250-020-00221-6" target="_blank">HTML Full Text</a>
                                                        
                                
                    <a href="">[PDF 196KB]</a>
                
                                <a href="https://link.springer.com/article/10.1007/s12250-020-00221-6" target="_blank">Springerlink</a>

                                                
                     
                     
                     
                
            </p>
        
         
                
            
                 
                            
        
                 
        <p class="citation">
            Wei Chen, Pengmian Feng, Kewei Liu, Meng Wu and Hao Lin. Computational Identification of Small Interfering RNA Targets in SARS-CoV-2[J]. Virologica Sinica, 2020, 35(3): 359-361. doi: 10.1007/s12250-020-00221-6.
        </p>
    </p><p class="article-list-zy morenstyle">
    		         
			<span class="search-article-abstract">With the epidemic of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) worldwide and in the absence of any effective vaccine, there is an urgent need to find a specific anti- SARS-CoV-2 agent. In this study, by analyzing the secondary structures of the SARS-CoV-2 genome (MN908947), several 21~25 base-long segments were obtained and selected as the potential targets of small interfering RNA duplexes. Moreover, it was also found that these targets are conserved among different strains. We hope the results will contribute to the pharmaceutical research and therapy of the SARS-CoV-2.</span>
            </p><p class="main-right fr">
	        	<p class="main-left">
     
    
        <p class="tit">Vol 35, No 3 (June, 2020)</p>
        <p>ISSN 1674-0769</p>
        <p>EISSN 1995-820X</p>
        <p>CN 42-1760/Q</p>
        <p>Editor in Chief: Zheng-Li Shi</p>
        <p>2019 Impact Factor 3.242</p>
        <p class="r">2019 Journal Citation Reports</p>
    
</p>


    <a href="https://mc03.manuscriptcentral.com/vs" class="Manuscript mb20" target="_blank">Submit Manuscript</a>

<p class="base-title base-title4">GALLERY</p>

                    <p class="item">
                <a href="/news/gallery_en/177351c7-5168-4de5-bc87-44d51cf40ed4_en.htm">
                                         
                    <p class="text">Inaugural issue. 1986(1), Virol Sin</p>
                </a>
            </p>
                            <p class="item">
                <a href="/news/gallery_en/2a635d24-87d6-4036-bf84-7a3b31ce00fd_en.htm">
                                         
                    <p class="text">Nipah virus distribution. 2018(5), Virol Sin</p>
                </a>
            </p>
                            <p class="item">
                <a href="/news/gallery_en/d717ca44-6135-4839-b256-f19d91c2f423_en.htm">
                                         
                    <p class="text">Host cytoskeleton in flavivirus infection. 2019(1), Virol Sin</p>
                </a>
            </p>
                            <p class="item">
                <a href="/news/gallery_en/3fef7296-032e-4eca-849b-f55654f635ce_en.htm">
                                         
                    <p class="text">ZIKV particle. 2019(2), Virol Sin</p>
                </a>
            </p>
                            <p class="item">
                <a href="/news/gallery_en/23f6a525-f9d6-4e32-890b-c41b8b60062c_en.htm">
                                         
                    <p class="text">Phage therapy. 2019(4), Virol Sin</p>
                </a>
            </p>
                            <p class="item">
                <a href="/news/gallery_en/c76fc7e1-5a0e-4ec1-b36e-48082a848207_en.htm">
                                         
                    <p class="text">Regulation of DENV infection. 2019(5), Virol Sin</p>
                </a>
            </p>
                            <p class="item">
                <a href="/news/gallery_en/664fbe2a-430c-4692-b4b2-962ce76e82b4_en.htm">
                                         
                    <p class="text">Increased endolysosomes in SFTSV infected SNX11-KO cells. 2019(5), Virol Sin</p>
                </a>
            </p>
                
    </p></div>